<!doctype html>
</body>
</html>
<html>
	<head>
		<title>Testing bibtex</title>
		<link rel="stylesheet" href="bib-publication-list.css"/>
	</head>
	<body>
    <noscript>
      <!-- bibtex source hidden by default, show it if JS disabled -->
      <style>
      #bibtex { display: block;}
      </style>
    </noscript>
		<table id="pubTable" class="display"></table>
        <pre id="bibtex" style="display:none;">
@article{Chen2014,
abstract = {Most vertebrate promoters lie in unmethylated CpG-dense islands, whereas methylation of the more sparsely distributed CpGs in the remainder of the genome is thought to contribute to transcriptional repression. Nonmethylated CG dinucleotides are recognized by CXXC finger protein 1 (CXXC1, also known as CFP1), which recruits SETD1A (also known as Set1) methyltransferase for trimethylation of histone H3 lysine 4, an active promoter mark. Genomic regions enriched for CpGs are thought to be either absent or irrelevant in invertebrates that lack DNA methylation, such as C. elegans; however, a CXXC1 ortholog (CFP-1) is present. Here we demonstrate that C. elegans CFP-1 targets promoters with high CpG density, and these promoters are marked by high levels of H3K4me3. Furthermore, as for mammalian promoters, high CpG content is associated with nucleosome depletion irrespective of transcriptional activity. We further show that highly occupied target (HOT) regions identified by the binding of a large number of transcription factors are CpG-rich promoters in C. elegans and human genomes, suggesting that the unusually high factor association at HOT regions may be a consequence of CpG-linked chromatin accessibility. Our results indicate that nonmethylated CpG-dense sequence is a conserved genomic signal that promotes an open chromatin state, targeting by a CXXC1 ortholog, and H3K4me3 modification in both C. elegans and human genomes.},
author = {Chen, Ron A-J and Stempor, Przemyslaw and Down, Thomas a. and Zeiser, Eva and Feuer, Sky K. and Ahringer, Julie},
doi = {10.1101/gr.161992.113},
file = {:Users/przemol/Documents/Mendeley Desktop/Genome Res.-2014-Chen-gr.161992.113.pdf:pdf},
issn = {1549-5469},
journal = {Genome research},
month = {jul},
number = {7},
pages = {1138--46},
pmid = {24653213},
title = {{Extreme HOT regions are CpG-dense promoters in C. elegans and humans.}},
url = {http://genome.cshlp.org/cgi/doi/10.1101/gr.161992.113 http://www.ncbi.nlm.nih.gov/pubmed/24653213 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4079969},
volume = {24},
year = {2014}
}
@article{Ho2014,
author = {Ho, Joshua W K and Jung, Youngsook L and Liu, Tao and Alver, Burak H and Lee, Soohyun and Ikegami, Kohta and Sohn, Kyung-ah and Minoda, Aki and Tolstorukov, Michael Y and Appert, Alex and Parker, Stephen C J and Gu, Tingting and Kundaje, Anshul and Riddle, Nicole C and Bishop, Eric and Egelhofer, Thea a and Hu, Shawn and Alekseyenko, Artyom a and Rechtsteiner, Andreas and Asker, Dalal and Belsky, Jason a and Bowman, Sarah K and Chen, Q Brent and Chen, Ron a and Day, Daniel S and Dong, Yan and Dose, Andrea C and Duan, Xikun and Epstein, Charles B and Ercan, Sevinc and Feingold, Elise a and Ferrari, Francesco and Garrigues, Jacob M and Gehlenborg, Nils and Good, Peter J and Haseley, Psalm and He, Daniel and Herrmann, Moritz and Hoffman, Michael M and Jeffers, Tess E and Kharchenko, Peter V and Kolasinska-zwierz, Paulina and Kotwaliwale, Chitra V and Kumar, Nischay and Langley, Sasha a and Larschan, Erica N and Latorre, Isabel and Libbrecht, Maxwell W and Lin, Xueqiu and Park, Richard and Pazin, Michael J and Pham, Hoang N and Plachetka, Annette and Qin, Bo and Schwartz, Yuri B and Shoresh, Noam and Stempor, Przemyslaw and Vielle, Anne and Wang, Chengyang and Whittle, Christina M and Xue, Huiling and Kingston, Robert E and Kim, Ju Han and Bernstein, Bradley E and Dernburg, Abby F and Pirrotta, Vincenzo and Kuroda, Mitzi I and Noble, William S and Tullius, Thomas D and Kellis, Manolis and Macalpine, David M and Strome, Susan and Elgin, Sarah C R and Ad, Nhdf},
doi = {10.1038/nature13415},
file = {:Users/przemol/Documents/Mendeley Desktop/nature13415.pdf:pdf},
issn = {14764687},
journal = {Nature},
keywords = {elegans,states},
number = {7515},
pages = {449--452},
pmid = {25164756},
publisher = {Nature Publishing Group},
title = {{Comparative analysis ofmetazoan chromatin organization}},
url = {http://dx.doi.org/10.1038/nature13415},
volume = {512},
year = {2014}
}
@misc{Stemor2014,
author = {Stemor, Przemyslaw},
doi = {10.5281/zenodo.11427},
month = {aug},
title = {{rBEADS - The R implementation of Bias Elimination Algorithm for Deep Sequencing}},
url = {https://github.com/przemol/rbeads},
year = {2014}
}
@article{Chen2013,
abstract = {RNA polymerase transcription initiation sites are largely unknown in Caenorhabditis elegans. The initial 5' end of most protein-coding transcripts is removed by trans-splicing, and noncoding initiation sites have not been investigated. We characterized the landscape of RNA Pol II transcription initiation, identifying 73,500 distinct clusters of initiation. Bidirectional transcription is frequent, with a peak of transcriptional pairing at 120 bp. We assign transcription initiation sites to 7691 protein-coding genes and find that they display features typical of eukaryotic promoters. Strikingly, the majority of initiation events occur in regions with enhancer-like chromatin signatures. Based on the overlap of transcription initiation clusters with mapped transcription factor binding sites, we define 2361 transcribed intergenic enhancers. Remarkably, productive transcription elongation across these enhancers is predominantly in the same orientation as that of the nearest downstream gene. Directed elongation from an upstream enhancer toward a downstream gene could potentially deliver RNA polymerase II to a proximal promoter, or alternatively might function directly as a distal promoter. Our results provide a new resource to investigate transcription regulation in metazoans.},
author = {Chen, Ron A-J and Down, Thomas A and Stempor, Przemyslaw and Chen, Q Brent and Egelhofer, Thea A and Hillier, LaDeana W and Jeffers, Tess E and Ahringer, Julie},
doi = {10.1101/gr.153668.112},
file = {:Users/przemol/Documents/Mendeley Desktop/Genome Res.-2013-Chen-1339-47.pdf:pdf},
issn = {1549-5469},
journal = {Genome research},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: metabolism,Caenorhabditis elegans: genetics,Caenorhabditis elegans: metabolism,Chromatin,Chromatin: genetics,DNA,Enhancer Elements,Genetic,Molecular Sequence Annotation,Promoter Regions,RNA Polymerase II,RNA Polymerase II: metabolism,Sequence Analysis,Transcription Initiation Site,Transcriptional Activation},
month = {aug},
number = {8},
pages = {1339--47},
pmid = {23550086},
title = {{The landscape of RNA polymerase II transcription initiation in C. elegans reveals promoter and enhancer architectures.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3730107{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {23},
year = {2013}
}
@article{Latorre2013,
author = {Latorre, Isabel and Chesney, Michael a and Garrigues, Jacob M and Stempor, Przemyslaw and Appert, Alex and Francesconi, Mirko and Strome, Susan and Ahringer, Julie},
doi = {10.1101/gad.255810.114.Freely},
file = {:Users/przemol/Documents/Mendeley Desktop/Genes Dev.-2015-Latorre-495-500.pdf:pdf},
pages = {495--500},
title = {{The DREAM complex promotes gene body H2A . Z for target repression}},
year = {2013}
}
@article{Stempor2012,
abstract = {BACKGROUND: MicroRNA (miRNA) directed gene repression is an important mechanism of posttranscriptional regulation. Comprehensive analyses of how microRNA influence biological processes requires paired miRNA-mRNA expression datasets. However, a review of both GEO and ArrayExpress repositories revealed few such datasets, which was in stark contrast to the large number of messenger RNA (mRNA) only datasets. It is of interest that numerous primary miRNAs (precursors of microRNA) are known to be co-expressed with coding genes (host genes). RESULTS: We developed a miRNA-mRNA interaction analyses pipeline. The proposed solution is based on two miRNA expression prediction methods - a scaling function and a linear model. Additionally, miRNA-mRNA anti-correlation analyses are used to determine the most probable miRNA gene targets (i.e. the differentially expressed genes under the influence of up- or down-regulated microRNA). Both the consistency and accuracy of the prediction method is ensured by the application of stringent statistical methods. Finally, the predicted targets are subjected to functional enrichment analyses including GO, KEGG and DO, to better understand the predicted interactions. CONCLUSIONS: The MMpred pipeline requires only mRNA expression data as input and is independent of third party miRNA target prediction methods. The method passed extensive numerical validation based on the binding energy between the mature miRNA and 3' UTR region of the target gene. We report that MMpred is capable of generating results similar to that obtained using paired datasets. For the reported test cases we generated consistent output and predicted biological relationships that will help formulate further testable hypotheses.},
author = {Stempor, Przemyslaw A and Cauchi, Michael and Wilson, Paul},
doi = {10.1186/1471-2164-13-620},
file = {:Users/przemol/Documents/Mendeley Desktop/1471-2164-13-620.pdf:pdf},
issn = {1471-2164},
journal = {BMC genomics},
keywords = {Chromosome Mapping,Computational Biology,Computational Biology: methods,Humans,Linear Models,Messenger,Messenger: genetics,Messenger: metabolism,MicroRNAs,MicroRNAs: genetics,MicroRNAs: metabolism,Oligonucleotide Array Sequence Analysis,RNA,Reproducibility of Results,Transcriptome},
month = {jan},
pages = {620},
pmid = {23151045},
title = {{MMpred: functional miRNA--mRNA interaction analyses by miRNA expression prediction.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3562514{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {13},
year = {2012}
}
@article{Vielle2012,
abstract = {The Caenorhabditis elegans dosage compensation complex (DCC) equalizes X-chromosome gene dosage between XO males and XX hermaphrodites by two-fold repression of X-linked gene expression in hermaphrodites. The DCC localizes to the X chromosomes in hermaphrodites but not in males, and some subunits form a complex homologous to condensin. The mechanism by which the DCC downregulates gene expression remains unclear. Here we show that the DCC controls the methylation state of lysine 20 of histone H4, leading to higher H4K20me1 and lower H4K20me3 levels on the X chromosomes of XX hermaphrodites relative to autosomes. We identify the PR-SET7 ortholog SET-1 and the Suv4-20 ortholog SET-4 as the major histone methyltransferases for monomethylation and di/trimethylation of H4K20, respectively, and provide evidence that X-chromosome enrichment of H4K20me1 involves inhibition of SET-4 activity on the X. RNAi knockdown of set-1 results in synthetic lethality with dosage compensation mutants and upregulation of X-linked gene expression, supporting a model whereby H4K20me1 functions with the condensin-like C. elegans DCC to repress transcription of X-linked genes. H4K20me1 is important for mitotic chromosome condensation in mammals, suggesting that increased H4K20me1 on the X may restrict access of the transcription machinery to X-linked genes via chromatin compaction.},
author = {Vielle, Anne and Lang, Jackie and Dong, Yan and Ercan, Sevinc and Kotwaliwale, Chitra and Rechtsteiner, Andreas and Appert, Alex and Chen, Q Brent and Dose, Andrea and Egelhofer, Thea and Kimura, Hiroshi and Stempor, Przemyslaw and Dernburg, Abby and Lieb, Jason D and Strome, Susan and Ahringer, Julie},
doi = {10.1371/journal.pgen.1002933},
file = {:Users/przemol/Documents/Mendeley Desktop/journal.pgen.1002933.pdf:pdf},
issn = {1553-7404},
journal = {PLoS genetics},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: genetics,Caenorhabditis elegans: genetics,Chromatin,Chromatin: genetics,Developmental,Disorders of Sex Development,Disorders of Sex Development: genetics,Dosage Compensation,Gene Expression Regulation,Genes,Genetic,Histone-Lysine N-Methyltransferase,Histone-Lysine N-Methyltransferase: genetics,Histone-Lysine N-Methyltransferase: metabolism,Histones,Histones: genetics,Male,Methylation,Methyltransferases,Methyltransferases: genetics,RNA Interference,X Chromosome,X Chromosome: genetics,X-Linked},
month = {sep},
number = {9},
pages = {e1002933},
pmid = {23028348},
title = {{H4K20me1 contributes to downregulation of X-linked genes for C. elegans dosage compensation.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3441679{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {8},
year = {2012}
}

</pre>
  	<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
		<script type="text/javascript" src="build/bib-list.js"></script>
		<script type="text/javascript">
		    $(document).ready(function() {
        	bibtexify("#bibtex", "pubTable", {'tweet': 'vkaravir'});
    		});
		</script>
	</body>
</html>
