<!doctype html>
</body>
</html>
<html>
	<head>
		<title>Testing bibtex</title>
		<link rel="stylesheet" href="bib-publication-list.css"/>
	</head>
	<body>
    <noscript>
      <!-- bibtex source hidden by default, show it if JS disabled -->
      <style>
      #bibtex { display: block;}
      </style>
    </noscript>
		<table id="pubTable" class="display"></table>
<pre id="bibtex" style="display:none;">

@article{Ho2014,
author = {Ho, Joshua W K and Jung, Youngsook L and Liu, Tao and Alver, Burak H and Lee, Soohyun and Ikegami, Kohta and Sohn, Kyung-ah and Minoda, Aki and Tolstorukov, Michael Y and Appert, Alex and Parker, Stephen C J and Gu, Tingting and Kundaje, Anshul and Riddle, Nicole C and Bishop, Eric and Egelhofer, Thea a and Hu, Shawn and Alekseyenko, Artyom a and Rechtsteiner, Andreas and Asker, Dalal and Belsky, Jason a and Bowman, Sarah K and Chen, Q Brent and Chen, Ron a and Day, Daniel S and Dong, Yan and Dose, Andrea C and Duan, Xikun and Epstein, Charles B and Ercan, Sevinc and Feingold, Elise a and Ferrari, Francesco and Garrigues, Jacob M and Gehlenborg, Nils and Good, Peter J and Haseley, Psalm and He, Daniel and Herrmann, Moritz and Hoffman, Michael M and Jeffers, Tess E and Kharchenko, Peter V and Kolasinska-zwierz, Paulina and Kotwaliwale, Chitra V and Kumar, Nischay and Langley, Sasha a and Larschan, Erica N and Latorre, Isabel and Libbrecht, Maxwell W and Lin, Xueqiu and Park, Richard and Pazin, Michael J and Pham, Hoang N and Plachetka, Annette and Qin, Bo and Schwartz, Yuri B and Shoresh, Noam and Stempor, Przemyslaw and Vielle, Anne and Wang, Chengyang and Whittle, Christina M and Xue, Huiling and Kingston, Robert E and Kim, Ju Han and Bernstein, Bradley E and Dernburg, Abby F and Pirrotta, Vincenzo and Kuroda, Mitzi I and Noble, William S and Tullius, Thomas D and Kellis, Manolis and Macalpine, David M and Strome, Susan and Elgin, Sarah C R and Ad, Nhdf},
doi = {10.1038/nature13415},
file = {:Users/przemol/Documents/Mendeley Desktop/nature13415.pdf:pdf},
issn = {14764687},
journal = {Nature},
keywords = {elegans,states},
number = {7515},
pages = {449--452},
pmid = {25164756},
publisher = {Nature Publishing Group},
title = {{Comparative analysis ofmetazoan chromatin organization}},
url = {http://dx.doi.org/10.1038/nature13415},
volume = {512},
year = {2014}
}
@article{Weick2014,
abstract = {Piwi-interacting RNAs (piRNA) are small regulatory RNAs with essential roles in maintaining genome integrity in animals and protists. Most Caenorhabditis elegans piRNAs are transcribed from two genomic clusters that likely contain thousands of individual transcription units; however, their biogenesis is not understood. Here we identify and characterize prde-1 (piRNA silencing-defective) as the first essential C. elegans piRNA biogenesis gene. Analysis of prde-1 provides the first direct evidence that piRNA precursors are 28- to 29-nucleotide (nt) RNAs initiating 2 nt upstream of mature piRNAs. PRDE-1 is a nuclear germline-expressed protein that localizes to chromosome IV. PRDE-1 is required specifically for the production of piRNA precursors from genomic loci containing an 8-nt upstream motif, the Ruby motif. The expression of a second class of motif-independent piRNAs is unaffected in prde-1 mutants. We exploited this finding to determine the targets of the motif-independent class of piRNAs. Together, our data provide new insights into both the biogenesis and function of piRNAs in gene regulation.},
author = {Weick, Eva-Maria Maria and Sarkies, Peter and Silva, Nicola and Chen, Ron a. and Moss, Sylviane M M and Cording, Amy C. and Ahringer, Julie and Martinez-Perez, Enrique and Miska, Eric a.},
doi = {10.1101/gad.238105.114},
file = {:Users/przemol/Documents/Mendeley Desktop/783.pdf:pdf},
isbn = {1549-5477 (Electronic)$\backslash$n0890-9369 (Linking)},
issn = {1549-5477},
journal = {Genes {\&} development},
keywords = {2014,21U-RNA,21u-rna,27,Amino Acid Motifs,Animals,C. elegans,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: genetics,Caenorhabditis elegans Proteins: metabolism,Caenorhabditis elegans: genetics,Caenorhabditis elegans: metabolism,Chromosomes,Chromosomes: genetics,Developmental,Fertility,Fertility: genetics,Gene Expression Regulation,Germ Cells,Germ Cells: physiology,Mutation,PRDE-1,RNA,RNA Stability,RNA Stability: genetics,Small Interfering,Small Interfering: biosynthesis,Small Interfering: chemistry,Small Interfering: genetics,c,elegans,for this article,have an evolutionarily,piRNA biogenesis,pirna biogenesis,pirnas,piwi-interacting rnas,prde-1,received january 21,revised version accepted february,supplemental material is available},
number = {7},
pages = {783--96},
pmid = {24696457},
title = {{PRDE-1 is a nuclear factor essential for the biogenesis of Ruby motif-dependent piRNAs in C. elegans.}},
url = {http://genesdev.cshlp.org/content/28/7/783.short$\backslash$nhttp://www.ncbi.nlm.nih.gov/pubmed/24696457$\backslash$nhttp://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=4015492{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {28},
year = {2014}
}
@article{Chen2014,
abstract = {Most vertebrate promoters lie in unmethylated CpG-dense islands, whereas methylation of the more sparsely distributed CpGs in the remainder of the genome is thought to contribute to transcriptional repression. Nonmethylated CG dinucleotides are recognized by CXXC finger protein 1 (CXXC1, also known as CFP1), which recruits SETD1A (also known as Set1) methyltransferase for trimethylation of histone H3 lysine 4, an active promoter mark. Genomic regions enriched for CpGs are thought to be either absent or irrelevant in invertebrates that lack DNA methylation, such as C. elegans; however, a CXXC1 ortholog (CFP-1) is present. Here we demonstrate that C. elegans CFP-1 targets promoters with high CpG density, and these promoters are marked by high levels of H3K4me3. Furthermore, as for mammalian promoters, high CpG content is associated with nucleosome depletion irrespective of transcriptional activity. We further show that highly occupied target (HOT) regions identified by the binding of a large number of transcription factors are CpG-rich promoters in C. elegans and human genomes, suggesting that the unusually high factor association at HOT regions may be a consequence of CpG-linked chromatin accessibility. Our results indicate that nonmethylated CpG-dense sequence is a conserved genomic signal that promotes an open chromatin state, targeting by a CXXC1 ortholog, and H3K4me3 modification in both C. elegans and human genomes.},
author = {Chen, Ron A-J and Stempor, Przemyslaw and Down, Thomas A and Zeiser, Eva and Feuer, Sky K and Ahringer, Julie},
doi = {10.1101/gr.161992.113},
file = {:Users/przemol/Documents/Mendeley Desktop/1138.pdf:pdf},
issn = {1549-5469},
journal = {Genome research},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans: genetics,Caenorhabditis elegans: metabolism,CpG Islands,DNA Methylation,Epigenesis, Genetic,Epigenomics,Gene Expression,Gene Expression Regulation,Gene Order,Genes, Reporter,Genetic Vectors,Genetic Vectors: genetics,Histones,Histones: metabolism,Humans,Nucleosomes,Nucleosomes: genetics,Nucleosomes: metabolism,Promoter Regions, Genetic,Protein Binding,Transcription Factors,Transcription Factors: metabolism},
number = {7},
pages = {1138--46},
pmid = {24653213},
title = {{Extreme HOT regions are CpG-dense promoters in C. elegans and humans.}},
url = {http://genome.cshlp.org/content/24/7/1138.long},
volume = {24},
year = {2014}
}
@article{Gaggioli2014,
abstract = {Cyclin-dependent kinase (CDK) plays a vital role in proliferation control across eukaryotes. Despite this, how CDK mediates cell cycle and developmental transitions in metazoa is poorly understood. In this paper, we identify orthologues of Sld2, a CDK target that is important for DNA replication in yeast, and characterize SLD-2 in the nematode worm Caenorhabditis elegans. We demonstrate that SLD-2 is required for replication initiation and the nuclear retention of a critical component of the replicative helicase CDC-45 in embryos. SLD-2 is a CDK target in vivo, and phosphorylation regulates the interaction with another replication factor, MUS-101. By mutation of the CDK sites in sld-2, we show that CDK phosphorylation of SLD-2 is essential in C. elegans. Finally, using a phosphomimicking sld-2 mutant, we demonstrate that timely CDK phosphorylation of SLD-2 is an important control mechanism to allow normal proliferation in the germline. These results determine an essential function of CDK in metazoa and identify a developmental role for regulated SLD-2 phosphorylation.},
author = {Gaggioli, Vincent and Zeiser, Eva and Rivers, David and Bradshaw, Charles R. and Ahringer, Julie and Zegerman, Philip},
doi = {10.1083/jcb.201310083},
file = {:Users/przemol/Documents/Mendeley Desktop/JCB{\_}201310083.pdf:pdf},
issn = {00219525},
journal = {Journal of Cell Biology},
number = {4},
pages = {507--522},
pmid = {24535824},
title = {{CDK phosphorylation of SLD-2 is required for replication initiation and germline development in C. elegans}},
volume = {204},
year = {2014}
}
@article{Chen2013,
abstract = {RNA polymerase transcription initiation sites are largely unknown in Caenorhabditis elegans. The initial 5' end of most protein-coding transcripts is removed by trans-splicing, and noncoding initiation sites have not been investigated. We characterized the landscape of RNA Pol II transcription initiation, identifying 73,500 distinct clusters of initiation. Bidirectional transcription is frequent, with a peak of transcriptional pairing at 120 bp. We assign transcription initiation sites to 7691 protein-coding genes and find that they display features typical of eukaryotic promoters. Strikingly, the majority of initiation events occur in regions with enhancer-like chromatin signatures. Based on the overlap of transcription initiation clusters with mapped transcription factor binding sites, we define 2361 transcribed intergenic enhancers. Remarkably, productive transcription elongation across these enhancers is predominantly in the same orientation as that of the nearest downstream gene. Directed elongation from an upstream enhancer toward a downstream gene could potentially deliver RNA polymerase II to a proximal promoter, or alternatively might function directly as a distal promoter. Our results provide a new resource to investigate transcription regulation in metazoans.},
author = {Chen, Ron a J and Down, Thomas a. and Stempor, Przemyslaw and Chen, Q. Brent and Egelhofer, Thea a. and Hillier, LaDeana W. and Jeffers, Tess E. and Ahringer, Julie},
doi = {10.1101/gr.153668.112},
file = {:Users/przemol/Documents/Mendeley Desktop/2013{\_}GenomeResearch.pdf:pdf},
isbn = {1549-5469 (Electronic)$\backslash$r1088-9051 (Linking)},
issn = {10889051},
journal = {Genome Research},
number = {8},
pages = {1339--1347},
pmid = {23550086},
title = {{The landscape of RNA polymerase II transcription initiation in C. elegans reveals promoter and enhancer architectures}},
volume = {23},
year = {2013}
}
@article{Woods2013,
author = {Woods, Shane and Coghlan, Avril and Rivers, David and Warnecke, Tobias and Jeffries, Sean J. and Kwon, Taejoon and Rogers, Anthony and Hurst, Laurence D. and Ahringer, Julie},
doi = {10.1371/journal.pgen.1003330},
file = {:Users/przemol/Documents/Mendeley Desktop/2013{\_}PLoSGenetics.pdf:pdf},
issn = {1553-7404},
journal = {PLoS Genetics},
number = {5},
pages = {e1003330},
title = {{Duplication and Retention Biases of Essential and Non-Essential Genes Revealed by Systematic Knockdown Analyses}},
url = {http://dx.plos.org/10.1371/journal.pgen.1003330},
volume = {9},
year = {2013}
}
@article{Stamper2013,
abstract = {Meiotic recombination, an essential aspect of sexual reproduction, is initiated by programmed DNA double-strand breaks (DSBs). DSBs are catalyzed by the widely-conserved Spo11 enzyme; however, the activity of Spo11 is regulated by additional factors that are poorly conserved through evolution. To expand our understanding of meiotic regulation, we have characterized a novel gene, dsb-1, that is specifically required for meiotic DSB formation in the nematode Caenorhabditis elegans. DSB-1 localizes to chromosomes during early meiotic prophase, coincident with the timing of DSB formation. DSB-1 also promotes normal protein levels and chromosome localization of DSB-2, a paralogous protein that plays a related role in initiating recombination. Mutations that disrupt crossover formation result in prolonged DSB-1 association with chromosomes, suggesting that nuclei may remain in a DSB-permissive state. Extended DSB-1 localization is seen even in mutants with defects in early recombination steps, including spo-11, suggesting that the absence of crossover precursors triggers the extension. Strikingly, failure to form a crossover precursor on a single chromosome pair is sufficient to extend the localization of DSB-1 on all chromosomes in the same nucleus. Based on these observations we propose a model for crossover assurance that acts through DSB-1 to maintain a DSB-permissive state until all chromosome pairs acquire crossover precursors. This work identifies a novel component of the DSB machinery in C. elegans, and sheds light on an important pathway that regulates DSB formation for crossover assurance.},
author = {Stamper, Ericca L. and Rodenbusch, Stacia E. and Rosu, Simona and Ahringer, Julie and Villeneuve, Anne M. and Dernburg, Abby F.},
doi = {10.1371/journal.pgen.1003679},
file = {:Users/przemol/Documents/Mendeley Desktop/2013{\_}PLoSGenetics2.pdf:pdf},
isbn = {1553-7404 (Electronic)$\backslash$r1553-7390 (Linking)},
issn = {15537390},
journal = {PLoS Genetics},
number = {8},
pages = {1--18},
pmid = {23990794},
title = {{Identification of DSB-1, a Protein Required for Initiation of Meiotic Recombination in Caenorhabditis elegans, Illuminates a Crossover Assurance Checkpoint}},
volume = {9},
year = {2013}
}
@article{Fievet2013,
abstract = {Although single-gene loss-of-function analyses can identify components of particular processes, important molecules are missed owing to the robustness of biological systems. Here we show that large-scale RNAi screening for suppression interactions with functionally related mutants greatly expands the repertoire of genes known to act in a shared process and reveals a new layer of functional relationships. We performed RNAi screens for 17 Caenorhabditis elegans cell polarity mutants, generating the most comprehensive polarity network in a metazoan, connecting 184 genes. Of these, 72{\%} were not previously linked to cell polarity and 80{\%} have human homologues. We experimentally confirmed functional roles predicted by the network and characterized through biophysical analyses eight myosin regulators. In addition, we discovered functional redundancy between two unknown polarity genes. Similar systematic genetic interaction screens for other biological processes will help uncover the inventory of relevant genes and their patterns of interactions.},
author = {Fievet, Bruno Thomas and Rodriguez, Josana and Naganathan, Sundar and Lee, Christine and Zeiser, Eva and Ishidate, Takao and Shirayama, Masaki and Grill, Stephan and Ahringer, Julie},
doi = {10.1038/ncb2639},
file = {:Users/przemol/Documents/Mendeley Desktop/2013{\_}NatureCellBiology.pdf:pdf},
isbn = {1476-4679 (Electronic)$\backslash$r1465-7392 (Linking)},
issn = {1476-4679},
journal = {Nature cell biology},
keywords = {Actomyosin,Actomyosin: metabolism,Animals,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: genetics,Caenorhabditis elegans Proteins: metabolism,Caenorhabditis elegans: cytology,Caenorhabditis elegans: genetics,Cell Polarity,Cell Polarity: genetics,Embryo, Nonmammalian,Embryo, Nonmammalian: cytology,Gene Knockdown Techniques,Gene Regulatory Networks,Genes, Helminth,Genes, Lethal,Molecular Sequence Annotation,Protein Kinases,Protein Kinases: genetics,Protein Kinases: metabolism,RNA Interference,Signal Transduction},
number = {1},
pages = {103--12},
pmid = {23242217},
publisher = {Nature Publishing Group},
title = {{Systematic genetic interaction screens uncover cell polarity regulators and functional redundancy.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3836181{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {15},
year = {2013}
}
@article{Vielle2012,
abstract = {The Caenorhabditis elegans dosage compensation complex (DCC) equalizes X-chromosome gene dosage between XO males and XX hermaphrodites by two-fold repression of X-linked gene expression in hermaphrodites. The DCC localizes to the X chromosomes in hermaphrodites but not in males, and some subunits form a complex homologous to condensin. The mechanism by which the DCC downregulates gene expression remains unclear. Here we show that the DCC controls the methylation state of lysine 20 of histone H4, leading to higher H4K20me1 and lower H4K20me3 levels on the X chromosomes of XX hermaphrodites relative to autosomes. We identify the PR-SET7 ortholog SET-1 and the Suv4-20 ortholog SET-4 as the major histone methyltransferases for monomethylation and di/trimethylation of H4K20, respectively, and provide evidence that X-chromosome enrichment of H4K20me1 involves inhibition of SET-4 activity on the X. RNAi knockdown of set-1 results in synthetic lethality with dosage compensation mutants and upregulation of X-linked gene expression, supporting a model whereby H4K20me1 functions with the condensin-like C. elegans DCC to repress transcription of X-linked genes. H4K20me1 is important for mitotic chromosome condensation in mammals, suggesting that increased H4K20me1 on the X may restrict access of the transcription machinery to X-linked genes via chromatin compaction.},
author = {Vielle, Anne and Lang, Jackie and Dong, Yan and Ercan, Sevinc and Kotwaliwale, Chitra and Rechtsteiner, Andreas and Appert, Alex and Chen, Q Brent and Dose, Andrea and Egelhofer, Thea and Kimura, Hiroshi and Stempor, Przemyslaw and Dernburg, Abby and Lieb, Jason D and Strome, Susan and Ahringer, Julie},
doi = {10.1371/journal.pgen.1002933},
file = {:Users/przemol/Documents/Mendeley Desktop/journal.pgen.1002933.pdf:pdf},
issn = {1553-7404},
journal = {PLoS genetics},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: genetics,Caenorhabditis elegans: genetics,Chromatin,Chromatin: genetics,Developmental,Disorders of Sex Development,Disorders of Sex Development: genetics,Dosage Compensation,Gene Expression Regulation,Genes,Genetic,Histone-Lysine N-Methyltransferase,Histone-Lysine N-Methyltransferase: genetics,Histone-Lysine N-Methyltransferase: metabolism,Histones,Histones: genetics,Male,Methylation,Methyltransferases,Methyltransferases: genetics,RNA Interference,X Chromosome,X Chromosome: genetics,X-Linked},
month = {sep},
number = {9},
pages = {e1002933},
pmid = {23028348},
title = {{H4K20me1 contributes to downregulation of X-linked genes for C. elegans dosage compensation.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3441679{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {8},
year = {2012}
}
@article{Liu2011,
author = {Liu, Tao and Rechtsteiner, Andreas and Egelhofer, Thea A and Vielle, Anne and Latorre, Isabel and Cheung, Ming-sin and Ercan, Sevinc and Ikegami, Kohta and Jensen, Morten and Kolasinska-zwierz, Paulina and Rosenbaum, Heidi and Shin, Hyunjin and Taing, Scott and Takasaki, Teruaki and Iniguez, A Leonardo and Desai, Arshad and Dernburg, Abby F and Kimura, Hiroshi and Lieb, Jason D and Ahringer, Julie and Strome, Susan and Liu, X Shirley},
doi = {10.1101/gr.115519.110.Freely},
file = {:Users/przemol/Documents/Mendeley Desktop/227.pdf:pdf},
pages = {227--236},
title = {{Broad chromosomal domains of histone modification patterns in C. elegans.pdf}},
year = {2011}
}
@article{Cheung2011,
abstract = {Genomic sequences obtained through high-throughput sequencing are not uniformly distributed across the genome. For example, sequencing data of total genomic DNA show significant, yet unexpected enrichments on promoters and exons. This systematic bias is a particular problem for techniques such as chromatin immunoprecipitation, where the signal for a target factor is plotted across genomic features. We have focused on data obtained from Illumina's Genome Analyser platform, where at least three factors contribute to sequence bias: GC content, mappability of sequencing reads, and regional biases that might be generated by local structure. We show that relying on input control as a normalizer is not generally appropriate due to sample to sample variation in bias. To correct sequence bias, we present BEADS (bias elimination algorithm for deep sequencing), a simple three-step normalization scheme that successfully unmasks real binding patterns in ChIP-seq data. We suggest that this procedure be done routinely prior to data interpretation and downstream analyses.},
author = {Cheung, Ming-Sin and Down, Thomas A and Latorre, Isabel and Ahringer, Julie},
doi = {10.1093/nar/gkr425},
file = {:Users/przemol/Documents/Mendeley Desktop/Cheung et al. - 2011 - Systematic bias in high-throughput sequencing data and its correction by BEADS(2).pdf:pdf},
issn = {1362-4962},
journal = {Nucleic acids research},
keywords = {Algorithms,Animals,Base Composition,Caenorhabditis elegans,Caenorhabditis elegans: genetics,Chromatin Immunoprecipitation,DNA, Helminth,DNA, Helminth: chemistry,High-Throughput Nucleotide Sequencing,High-Throughput Nucleotide Sequencing: methods,Sequence Analysis, DNA,Sequence Analysis, DNA: methods},
month = {aug},
number = {15},
pages = {e103},
pmid = {21646344},
title = {{Systematic bias in high-throughput sequencing data and its correction by BEADS.}},
url = {http://nar.oxfordjournals.org/content/early/2011/06/06/nar.gkr425.full},
volume = {39},
year = {2011}
}
@article{Zeiser2011,
author = {Zeiser, Eva and Fr{\o}kj{\ae}r-Jensen, Christian and Jorgensen, Erik and Ahringer, Julie},
doi = {10.1371/journal.pone.0020082},
file = {:Users/przemol/Documents/Mendeley Desktop/2011{\_}PlosOne.pdf:pdf},
issn = {1932-6203},
journal = {PLoS ONE},
number = {5},
pages = {e20082},
title = {{MosSCI and Gateway Compatible Plasmid Toolkit for Constitutive and Inducible Expression of Transgenes in the C. elegans Germline}},
url = {http://dx.plos.org/10.1371/journal.pone.0020082},
volume = {6},
year = {2011}
}
@article{Poulin2010,
abstract = {Attenuation of the EGFR (Epidermal Growth Factor Receptor) signalling cascade is crucial to control cell fate during development. A candidate-based RNAi approach in C. elegans identified CDT-2 as an attenuator of LET-23 (EGFR) signalling. Human CDT2 is a component of the conserved CDT2/CUL4/DDB1 ubiquitin ligase complex that plays a critical role in DNA replication and G2/M checkpoint. Within this complex, CDT2 is responsible for substrate recognition. This ubiquitin ligase complex has been shown in various organisms, including C. elegans, to target the replication-licensing factor CDT1, and the CDK inhibitor p21. However, no previous link to EGFR signalling has been identified.},
author = {Poulin, Gino B and Ahringer, Julie},
doi = {10.1186/1471-213X-10-109},
file = {:Users/przemol/Documents/Mendeley Desktop/2010{\_}BMC{\_}Dev{\_}Biol.pdf:pdf},
issn = {1471-213X},
journal = {BMC developmental biology},
month = {jan},
pages = {109},
pmid = {20977703},
title = {{The caenorhabditis elegans CDT-2 ubiquitin ligase is required for attenuation of EGFR signalling in vulva precursor cells.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2984460{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {10},
year = {2010}
}
@article{Gerstein2010,
author = {Gerstein, M. B. and Lu, Z. J. and {Van Nostrand}, E. L. and Cheng, C. and Arshinoff, B. I. and Liu, T. and Yip, K. Y. and Robilotto, R. and Rechtsteiner, a. and Ikegami, K. and Alves, P. and Chateigner, a. and Perry, M. and Morris, M. and Auerbach, R. K. and Feng, X. and Leng, J. and Vielle, a. and Niu, W. and Rhrissorrakrai, K. and Agarwal, a. and Alexander, R. P. and Barber, G. and Brdlik, C. M. and Brennan, J. and Brouillet, J. J. and Carr, a. and Cheung, M.-S. and Clawson, H. and Contrino, S. and Dannenberg, L. O. and Dernburg, a. F. and Desai, a. and Dick, L. and Dose, a. C. and Du, J. and Egelhofer, T. and Ercan, S. and Euskirchen, G. and Ewing, B. and Feingold, E. a. and Gassmann, R. and Good, P. J. and Green, P. and Gullier, F. and Gutwein, M. and Guyer, M. S. and Habegger, L. and Han, T. and Henikoff, J. G. and Henz, S. R. and Hinrichs, a. and Holster, H. and Hyman, T. and Iniguez, a. L. and Janette, J. and Jensen, M. and Kato, M. and Kent, W. J. and Kephart, E. and Khivansara, V. and Khurana, E. and Kim, J. K. and Kolasinska-Zwierz, P. and Lai, E. C. and Latorre, I. and Leahey, a. and Lewis, S. and Lloyd, P. and Lochovsky, L. and Lowdon, R. F. and Lubling, Y. and Lyne, R. and MacCoss, M. and Mackowiak, S. D. and Mangone, M. and McKay, S. and Mecenas, D. and Merrihew, G. and Miller, D. M. and Muroyama, a. and Murray, J. I. and Ooi, S.-L. and Pham, H. and Phippen, T. and Preston, E. a. and Rajewsky, N. and Ratsch, G. and Rosenbaum, H. and Rozowsky, J. and Rutherford, K. and Ruzanov, P. and Sarov, M. and Sasidharan, R. and Sboner, a. and Scheid, P. and Segal, E. and Shin, H. and Shou, C. and Slack, F. J. and Slightam, C. and Smith, R. and Spencer, W. C. and Stinson, E. O. and Taing, S. and Takasaki, T. and Vafeados, D. and Voronina, K. and Wang, G. and Washington, N. L. and Whittle, C. M. and Wu, B. and Yan, K.-K. and Zeller, G. and Zha, Z. and Zhong, M. and Zhou, X. and Ahringer, J. and Strome, S. and Gunsalus, K. C. and Micklem, G. and Liu, X. S. and Reinke, V. and Kim, S. K. and Hillier, L. W. and Henikoff, S. and Piano, F. and Snyder, M. and Stein, L. and Lieb, J. D. and Waterston, R. H.},
doi = {10.1126/science.1196914},
file = {:Users/przemol/Documents/Mendeley Desktop/2010{\_}Science{\_}330{\_}1775.pdf:pdf},
issn = {0036-8075},
journal = {Science},
month = {dec},
number = {6012},
pages = {1775--1787},
title = {{Integrative Analysis of the Caenorhabditis elegans Genome by the modENCODE Project}},
url = {http://www.sciencemag.org/cgi/doi/10.1126/science.1196914},
volume = {330},
year = {2010}
}
@article{Egelhofer2010,
abstract = {We have tested the specificity and utility of more than 200 antibodies raised against 57 different histone modifications in Drosophila melanogaster, Caenorhabditis elegans and human cells. Although most antibodies performed well, more than 25{\%} failed specificity tests by dot blot or western blot. Among specific antibodies, more than 20{\%} failed in chromatin immunoprecipitation experiments. We advise rigorous testing of histone-modification antibodies before use, and we provide a website for posting new test results (http://compbio.med.harvard.edu/antibodies/).},
author = {Egelhofer, Thea a and Minoda, Aki and Klugman, Sarit and Lee, Kyungjoon and Kolasinska-Zwierz, Paulina and Alekseyenko, Artyom a and Cheung, Ming-Sin and Day, Daniel S and Gadel, Sarah and Gorchakov, Andrey a and Gu, Tingting and Kharchenko, Peter V and Kuan, Samantha and Latorre, Isabel and Linder-Basso, Daniela and Luu, Ying and Ngo, Queminh and Perry, Marc and Rechtsteiner, Andreas and Riddle, Nicole C and Schwartz, Yuri B and Shanower, Gregory a and Vielle, Anne and Ahringer, Julie and Elgin, Sarah C R and Kuroda, Mitzi I and Pirrotta, Vincenzo and Ren, Bing and Strome, Susan and Park, Peter J and Karpen, Gary H and Hawkins, R David and Lieb, Jason D},
doi = {10.1038/nsmb.1972},
file = {:Users/przemol/Documents/Mendeley Desktop/2010{\_}Nat{\_}Struct{\_}Mol{\_}Biol.pdf:pdf},
issn = {1545-9985},
journal = {Nature structural {\&} molecular biology},
month = {dec},
number = {December},
pages = {8--11},
pmid = {21131980},
title = {{An assessment of histone-modification antibody quality.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/21131980},
year = {2010}
}
@article{StJohnston2010,
author = {{St Johnston}, Daniel and Ahringer, Julie},
doi = {10.1016/j.cell.2010.05.011},
file = {:Users/przemol/Documents/Mendeley Desktop/2010{\_}Cell.pdf:pdf},
issn = {00928674},
journal = {Cell},
month = {may},
number = {5},
pages = {757--774},
title = {{Cell Polarity in Eggs and Epithelia: Parallels and Diversity}},
url = {http://linkinghub.elsevier.com/retrieve/pii/S0092867410005477},
volume = {141},
year = {2010}
}
@article{Kolasinska-Zwierz2009,
abstract = {Variation in patterns of methylations of histone tails reflects and modulates chromatin structure and function. To provide a framework for the analysis of chromatin function in Caenorhabditis elegans, we generated a genome-wide map of histone H3 tail methylations. We find that C. elegans genes show distributions of histone modifications that are similar to those of other organisms, with H3K4me3 near transcription start sites, H3K36me3 in the body of genes and H3K9me3 enriched on silent genes. We also observe a novel pattern: exons are preferentially marked with H3K36me3 relative to introns. H3K36me3 exon marking is dependent on transcription and is found at lower levels in alternatively spliced exons, supporting a splicing-related marking mechanism. We further show that the difference in H3K36me3 marking between exons and introns is evolutionarily conserved in human and mouse. We propose that H3K36me3 exon marking in chromatin provides a dynamic link between transcription and splicing.},
author = {Kolasinska-Zwierz, Paulina and Down, Thomas and Latorre, Isabel and Liu, Tao and Liu, X Shirley and Ahringer, Julie},
doi = {10.1038/ng.322},
file = {:Users/przemol/Documents/Mendeley Desktop/2009{\_}NatGen{\_}41{\_}376.pdf:pdf},
isbn = {1546-1718 (Electronic)$\backslash$r1061-4036 (Linking)},
issn = {1061-4036},
journal = {Nature genetics},
number = {3},
pages = {376--381},
pmid = {19182803},
title = {{Differential chromatin marking of introns and expressed exons by H3K36me3.}},
volume = {41},
year = {2009}
}
@article{Panbianco2008,
abstract = {Spindle positioning is an essential feature of asymmetric cell division. The conserved PAR proteins together with heterotrimeric G proteins control spindle positioning in animal cells, but how these are linked is not known. In C. elegans, PAR protein activity leads to asymmetric spindle placement through cortical asymmetry of Galpha regulators GPR-1/2. Here, we establish that the casein kinase 1 gamma CSNK-1 and a PIP(2) synthesis enzyme (PPK-1) transduce PAR polarity to asymmetric Galpha regulation. PPK-1 is posteriorly enriched in the one-celled embryo through PAR and CSNK-1 activities. Loss of CSNK-1 causes uniformly high PPK-1 levels, high symmetric cortical levels of GPR-1/2 and LIN-5, and increased spindle pulling forces. In contrast, knockdown of ppk-1 leads to low GPR-1/2 levels and decreased spindle forces. Furthermore, loss of CSNK-1 leads to increased levels of PIP(2). We propose that asymmetric generation of PIP(2) by PPK-1 directs the posterior enrichment of GPR-1/2 and LIN-5, leading to posterior spindle displacement.},
author = {Panbianco, Costanza and Weinkove, David and Zanin, Esther and Jones, David and Divecha, Nullin and Gotta, Monica and Ahringer, Julie},
doi = {10.1016/j.devcel.2008.06.002},
file = {:Users/przemol/Documents/Mendeley Desktop/2008{\_}DevCell{\_}15{\_}198.pdf:pdf},
isbn = {1534-5807 (Print)},
issn = {1878-1551},
journal = {Developmental cell},
keywords = {5-Diphosphate,5-Diphosphate: metabolism,Animals,Biological,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: metabolism,Caenorhabditis elegans: cytology,Caenorhabditis elegans: embryology,Caenorhabditis elegans: enzymology,Casein Kinase I,Casein Kinase I: metabolism,Cell Nucleus,Cell Nucleus: enzymology,Embryo,Mitotic Spindle Apparatus,Mitotic Spindle Apparatus: enzymology,Models,Nonmammalian,Nonmammalian: cytology,Nonmammalian: enzymology,Phosphatidylinositol 4,Phosphotransferases (Alcohol Group Acceptor),Phosphotransferases (Alcohol Group Acceptor): meta,Protein Transport,Protein-Serine-Threonine Kinases,Protein-Serine-Threonine Kinases: metabolism,RNA Interference},
number = {2},
pages = {198--208},
pmid = {18694560},
title = {{A casein kinase 1 and PAR proteins regulate asymmetry of a PIP(2) synthesis enzyme for asymmetric spindle positioning.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2686839{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {15},
year = {2008}
}
@article{Rivers2008,
abstract = {Cell cycle lengths vary widely among different cells within an animal, yet mechanisms of cell cycle length regulation are poorly understood. In the Caenorhabditis elegans embryo, the first cell division produces two cells with different cell cycle lengths, which are dependent on the conserved partitioning-defective (PAR) polarity proteins. We show that two key cell cycle regulators, the Polo-like kinase PLK-1 and the cyclin-dependent kinase phosphatase CDC-25.1, are asymmetrically distributed in early embryos. PLK-1 shows anterior cytoplasmic enrichment and CDC-25.1 shows PLK-1-dependent enrichment in the anterior nucleus. Both proteins are required for normal mitotic progression. Furthermore, these asymmetries are controlled by PAR proteins and the muscle excess (MEX) proteins MEX-5/MEX-6, and the latter is linked to protein degradation. Our results support a model whereby the PAR and MEX-5/MEX-6 proteins asymmetrically control PLK-1 levels, which asymmetrically regulates CDC-25.1 to promote differences in cell cycle lengths. We suggest that control of Plk1 and Cdc25 may be relevant to regulation of cell cycle length in other developmental contexts.},
author = {Rivers, David M and Moreno, Sergio and Abraham, Mary and Ahringer, Julie},
doi = {10.1083/jcb.200710018},
file = {:Users/przemol/Documents/Mendeley Desktop/2008{\_}JCellBiol{\_}180{\_}877.pdf:pdf},
isbn = {1540-8140 (Electronic)$\backslash$n0021-9525 (Linking)},
issn = {1540-8140},
journal = {The Journal of cell biology},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: genetics,Caenorhabditis elegans Proteins: metabolism,Caenorhabditis elegans: genetics,Caenorhabditis elegans: metabolism,Cell Cycle Proteins,Cell Cycle Proteins: genetics,Cell Division,Cell Division: genetics,Cell Nucleus,Cell Nucleus: genetics,Cell Nucleus: metabolism,Cell Nucleus: ultrastructure,Cell Polarity,Cell Polarity: genetics,Cytokinesis,Cytokinesis: genetics,Embryo, Nonmammalian,Embryo, Nonmammalian: cytology,Embryo, Nonmammalian: metabolism,Embryonic Development,Embryonic Development: genetics,Protein-Serine-Threonine Kinases,Protein-Serine-Threonine Kinases: genetics,Protein-Serine-Threonine Kinases: metabolism,ras-GRF1,ras-GRF1: genetics,ras-GRF1: metabolism},
number = {5},
pages = {877--85},
pmid = {18316412},
title = {{PAR proteins direct asymmetry of the cell cycle regulators Polo-like kinase and Cdc25.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2265398{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {180},
year = {2008}
}
@article{Boutros2008,
author = {Boutros, Michael and Ahringer, Julie},
doi = {10.1038/nrg2364},
file = {:Users/przemol/Documents/Mendeley Desktop/2008{\_}NatRevGenet{\_}9{\_}554.pdf:pdf},
issn = {1471-0056},
journal = {Nature Reviews Genetics},
number = {7},
pages = {554--566},
title = {{The art and design of genetic screens: RNA interference}},
url = {http://www.nature.com/doifinder/10.1038/nrg2364},
volume = {9},
year = {2008}
}
@article{Dong2007,
author = {Dong, Yan and Bogdanova, Aliona and Habermann, Bianca and Zachariae, Wolfgang and Ahringer, Julie},
doi = {10.1186/1471-213X-7-19},
file = {:Users/przemol/Documents/Mendeley Desktop/2007{\_}BMCDevBiol{\_}20{\_}7{\_}19.pdf:pdf},
issn = {1471213X},
journal = {BMC Developmental Biology},
number = {1},
pages = {19},
title = {{Identification of the C. elegans anaphase promoting complex subunit Cdc26 by phenotypic profiling and functional rescue in yeast}},
url = {http://www.biomedcentral.com/1471-213X/7/19},
volume = {7},
year = {2007}
}
@article{Welchman2007,
abstract = {During animal development, a complex of Par3, Par6 and atypical protein kinase C (aPKC) plays a central role in cell polarisation. The small G protein Cdc42 also functions in cell polarity and has been shown in some cases to act by regulating the Par3 complex. However, it is not yet known whether Cdc42 and the Par3 complex widely function together in development or whether they have independent functions. For example, many studies have implicated Cdc42 in cell migrations, but the Par3 complex has only been little studied, with conflicting results. Here we examine the requirements for CDC-42 and the PAR-3/PAR-6/PKC-3 complex in a range of different developmental events. We found similar requirements in all tissues examined, including polarised growth of vulval precursors and seam cells, migrations of neuroblasts and axons, and the development of the somatic gonad. We also propose a novel role for primordial germ cells in mediating coalescence of the Caenorhabditis elegans gonad. These results indicate that CDC-42 and the PAR-3/PAR-6/aPKC complex function together in diverse cell types. ?? 2007 Elsevier Inc. All rights reserved.},
author = {Welchman, David P. and Mathies, Laura D. and Ahringer, Julie},
doi = {10.1016/j.ydbio.2007.02.022},
file = {:Users/przemol/Documents/Mendeley Desktop/2007{\_}DevBiol{\_}305{\_}347.pdf:pdf},
isbn = {0012-1606 (Print)$\backslash$r0012-1606 (Linking)},
issn = {00121606},
journal = {Developmental Biology},
keywords = {Anchor cell,Caenorhabditis elegans,Cdc42,Cell migration,Gonad,Par3,Par6,Polarity,Vulva,aPKC},
pages = {347--357},
pmid = {17383625},
title = {{Similar requirements for CDC-42 and the PAR-3/PAR-6/PKC-3 complex in diverse cell types}},
volume = {305},
year = {2007}
}
@article{Tsai2007,
author = {Tsai, M.-C. and Ahringer, J.},
doi = {10.1083/jcb.200708101},
file = {:Users/przemol/Documents/Mendeley Desktop/2007{\_}JCellBiol{\_}179{\_}397.pdf:pdf},
issn = {0021-9525},
journal = {The Journal of Cell Biology},
number = {3},
pages = {397--402},
title = {{Microtubules are involved in anterior-posterior axis formation in C. elegans embryos}},
url = {http://www.jcb.org/cgi/doi/10.1083/jcb.200708101},
volume = {179},
year = {2007}
}
@article{Ahringer2007,
author = {Ahringer, Eric A Miska {\&} Julie},
file = {:Users/przemol/Documents/Mendeley Desktop/2007{\_}NatBiotech{\_}25{\_}302.pdf:pdf},
journal = {Nature Biotechnology},
number = {3},
pages = {302--303},
title = {{RNA interference has second$\backslash$rhelpings}},
volume = {25},
year = {2007}
}
@article{Andrews2007,
abstract = {BACKGROUND: Endocytosis is involved in the regulation of many cellular events, including signalling, cell migration, and cell polarity. To begin to investigate roles for endocytosis in early C. elegans development, we examined the distribution and dynamics of early endosomes (EEs) in embryos.$\backslash$n$\backslash$nMETHODOLOGY/PRINCIPAL FINDINGS: EEs are primarily found at the cell periphery with an initially uniform distribution after fertilization. Strikingly, we find that during the first cell cycle, EEA-1 positive EEs become enriched at the anterior cortex. In contrast, the Golgi compartment shows no asymmetry in distribution. Asymmetric enrichment of EEs depends on acto-myosin contractility and embryonic PAR polarity. In addition to their localization at the cortex, EEs are also found around the centrosome. These EEs move rapidly (1.3 microm/s) from the cortex directly to the centrosome, a speed comparable to that of the minus end directed motor dynein.$\backslash$n$\backslash$nCONCLUSIONS/SIGNIFICANCE: We speculate that the asymmetry of early endosomes might play a role in cell asymmetries or fate decisions.},
author = {Andrews, Robert and Ahringer, Julie},
doi = {10.1371/journal.pone.0000493},
file = {:Users/przemol/Documents/Mendeley Desktop/2007{\_}PLoSOne{\_}2{\_}e493.pdf:pdf},
isbn = {1932-6203 (Electronic)},
issn = {1932-6203},
journal = {PloS one},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: physiology,Caenorhabditis elegans: physiology,Cell Cycle,Cell Polarity,Centrosome,Embryo, Nonmammalian,Embryo, Nonmammalian: cytology,Embryo, Nonmammalian: metabolism,Endocytosis,Endocytosis: physiology,Endosomes,Endosomes: metabolism,Fluorescent Antibody Technique,Gene Expression Regulation, Developmental},
number = {6},
pages = {e493},
pmid = {17551574},
title = {{Asymmetry of early endosome distribution in C. elegans embryos.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1876258{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {2},
year = {2007}
}
@article{Poulin2005,
abstract = {In Caenorhabditis elegans, numerous 'synMuv' (synthetic multivulval) genes encode for chromatin-associated proteins involved in transcriptional repression, including an orthologue of Rb and components of the NuRD histone deacetylase complex. These genes antagonize Ras signalling to prevent erroneous adoption of vulval fate. To identify new components of this mechanism, we performed a genome-wide RNA interference (RNAi) screen. After RNAi of 16 757 genes, we found nine new synMuv genes. Based on predicted functions and genetic epistasis experiments, we propose that at least four post-translational modifications converge to inhibit Ras-stimulated vulval development: sumoylation, histone tail deacetylation, methylation, and acetylation. In addition, we demonstrate a novel role for sumoylation in inhibiting LIN-12/Notch signalling in the vulva. We further show that many of the synMuv genes are involved in gene regulation outside the vulva, negatively regulating the expression of the Delta homologue lag-2. As most of the genes identified in this screen are conserved in humans, we suggest that similar interactions may be relevant in mammals for control of Ras and Notch signalling, crosstalk between these pathways, and cell proliferation.},
author = {Poulin, Gino and Dong, Yan and Fraser, Andrew G and Hopper, Neil a and Ahringer, Julie},
doi = {10.1038/sj.emboj.7600955},
file = {:Users/przemol/Documents/Mendeley Desktop/2005{\_}EMBOJ{\_}24{\_}2613.pdf:pdf},
isbn = {0261-4189; 0261-4189},
issn = {0261-4189},
journal = {The EMBO journal},
keywords = {c,chromatin remodelling,elegans,ras signalling},
number = {14},
pages = {2613--2623},
pmid = {15990876},
title = {{Chromatin regulation and sumoylation in the inhibition of Ras-induced vulval development in Caenorhabditis elegans.}},
volume = {24},
year = {2005}
}
@article{Pocock2004,
author = {Pocock, R.},
doi = {10.1242/dev.01110},
file = {:Users/przemol/Documents/Mendeley Desktop/2004{\_}Dev{\_}131{\_}2373.pdf:pdf},
issn = {0950-1991},
journal = {Development},
keywords = {caenorhabditis elegans,dorsal,evenskipped,intercalation,t-box genes},
number = {10},
pages = {2373--2385},
title = {{A regulatory network of T-box genes and the even-skipped homologue vab-7 controls patterning and morphogenesis in C. elegans}},
url = {http://dev.biologists.org/cgi/doi/10.1242/dev.01110},
volume = {131},
year = {2004}
}
@article{Kemp2004,
abstract = {Centrosomes are major determinants of mitotic spindle structure, but the mechanisms regulating their behavior remain poorly understood. The spd-2 gene of C. elegans is required for centrosome assembly or "maturation." Here we show that spd-2 encodes a coiled-coil protein that localizes within pericentriolar material (PCM) and in the immediate vicinity of centrioles. During maturation, SPD-2 gradually accumulates at the centrosome in a manner that is partially dependent on Aurora-A kinase and cytoplasmic dynein. Interestingly, SPD-2 interacts genetically with dynein heavy chain and SPD-5, another coiled-coil protein required for centrosome maturation. SPD-2 and SPD-5 are codependent for localization to the PCM, but SPD-2 localizes to centrioles independently of SPD-5. Surprisingly, we also find that SPD-2 is required for centrosome duplication and genetically interacts with ZYG-1, a kinase required for duplication. Thus, we have identified SPD-2 as a factor critical for the two basic functions of the centrosome-microtubule organization and duplication. ?? 2004 by Cell Press.},
author = {Kemp, Catherine a. and Kopish, Kevin R. and Zipperlen, Peder and Ahringer, Julie and O'Connell, Kevin F.},
doi = {10.1016/S1534-5807(04)00066-8},
file = {:Users/przemol/Documents/Mendeley Desktop/2004{\_}DevCell{\_}6{\_}511.pdf:pdf},
isbn = {1534-5807 (Print)$\backslash$r1534-5807 (Linking)},
issn = {15345807},
journal = {Developmental Cell},
number = {4},
pages = {511--523},
pmid = {15068791},
title = {{Centrosome maturation and duplication in C. elegans require the coiled-coil protein SPD-2}},
volume = {6},
year = {2004}
}
@article{Poulin2004,
abstract = {Genes linked to human cancers often function in evolutionary conserved pathways, and research in C. elegans has been instrumental in dissecting some of the pathways affected, such as apoptosis and Ras signalling. The advent of RNA interference (RNAi) technology has allowed high-throughput loss-of-function analyses of C. elegans gene functions. Here we review some of the most recent genome-wide RNAi screens that have been conducted and discuss their impact on cancer research and possibilities for future screens. We also show that genes causally implicated in human cancers are significantly more likely to have a C. elegans homologue than average, validating the use of C. elegans as a cancer gene discovery platform. We foresee that genome-wide RNAi screens in C. elegans will continue to be productive in identifying new cancer gene candidates and will provide further insights into cancer gene functions.},
author = {Poulin, Gino and Nandakumar, Ramkumar and Ahringer, Julie},
doi = {10.1038/sj.onc.1208010},
file = {:Users/przemol/Documents/Mendeley Desktop/2004{\_}Oncogene{\_}23{\_}8340.pdf:pdf},
isbn = {0950-9232 (Print)},
issn = {0950-9232},
journal = {Oncogene},
keywords = {apoptosis,c,cancer,elegans,elegans does not form,mutant phenotypes are clearly,other types of,ras,relevant for cancer,rnai,synthetic interaction,tumours per se},
number = {51},
pages = {8340--8345},
pmid = {15517014},
title = {{Genome-wide RNAi screens in Caenorhabditis elegans: impact on cancer research.}},
volume = {23},
year = {2004}
}
@article{Rappleye2003,
abstract = {BACKGROUND: Cell polarity is essential for many decisions made during development. While investigation of polarity-specific factors has yielded great insights into the polarization process, little is known on how these polarity-specific factors link to the basic cellular mechanisms that function in non-polarity aspects of the cell. To better understand the mechanisms that establish embryonic polarity, we investigated genes required for polarity in the one-cell C. elegans embryo that are also required for other non-polarity functions. This has led to the identification of the Pod-class of mutants that are characterized by osmosensitive embryos and defects in anterior-posterior polarity. RESULTS: Mutation in either of two loci of this class, emb-8 and pod-2, disrupts embryonic polarization and results in osmotically-sensitive embryos. Loss of emb-8, a previously uncharacterized polarity gene, causes mislocalization of PAR-3 and PAR-2 that molecularly mark the anterior and posterior cortices. emb-8 encodes NADPH-cytochrome P450 reductase, a protein supplying electrons to cytochrome P450-family enzymes, some of which catalyze fatty acid modifications. Cloning of the previously characterized polarity gene pod-2 reveals it encodes acetyl-CoA carboxylase, an enzyme that catalyzes the first step in de novo fatty acid synthesis. Depletion of fatty acid synthase, the next enzyme in the biosynthetic pathway, by RNA-interference (RNAi) also causes similar loss of one-cell polarity. Furthermore, pod-2 polarity defects can be rescued by addition of exogenous fatty acids. By following the behavior of the pronucleus in emb-8 and pod-2 mutant embryos, we demonstrate that loss of polarity correlates with impaired interaction between the pronucleus-centrosome complex and the posterior cortex. CONCLUSIONS: The characterization of emb-8 and pod-2 mutant embryos suggests that the pronucleus-centrosome complex interaction with the cortex plays a direct role in establishing polarity and that fatty acid pathways are important for this polarizing event.},
author = {Rappleye, Chad a and Tagawa, Akiko and {Le Bot}, Nathalie and Ahringer, Julie and Aroian, Raffi V},
doi = {10.1186/1471-213X-3-8},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}BMCDB{\_}3{\_}8.pdf:pdf},
isbn = {1471-213X (Electronic)},
issn = {1471-213X},
journal = {BMC developmental biology},
pages = {8},
pmid = {14527340},
title = {{Involvement of fatty acid pathways and cortical interaction of the pronuclear complex in Caenorhabditis elegans embryonic polarity.}},
volume = {3},
year = {2003}
}
@article{Malone2003,
abstract = {The centrosome and nucleus are intimately associated in most animal cells, yet the significance of this interaction is unknown. Mutations in the zyg-12 gene of Caenorhabditis elegans perturb the attachment of the centrosome to the nucleus, giving rise to aberrant spindles and ultimately, DNA segregation defects and lethality. These phenotypes indicate that the attachment is essential. ZYG-12 is a member of the Hook family of cytoskeletal linker proteins and localizes to both the nuclear envelope (via SUN-1) and centrosomes. ZYG-12 is able to bind the dynein subunit DLI-1 in a two-hybrid assay and is required for dynein localization to the nuclear envelope. Loss of dynein function causes a low percentage of defective centrosome/nuclei interactions in both Drosophila and Caenorhabditis elegans. We propose that dynein and ZYG-12 move the centrosomes toward the nucleus, followed by a ZYG-12/SUN-1-dependent anchorage.},
author = {Malone, Christian J. and Misner, Lisa and {Le Bot}, Nathalie and Tsai, Miao Chih and Campbell, Jay M. and Ahringer, Julie and White, John G.},
doi = {10.1016/S0092-8674(03)00985-1},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}Cell{\_}115{\_}825.pdf:pdf},
isbn = {0092-8674 (Print)$\backslash$r0092-8674 (Linking)},
issn = {00928674},
journal = {Cell},
number = {7},
pages = {825--836},
pmid = {14697201},
title = {{The C. elegans Hook Protein, ZYG-12, Mediates the Essential Attachment between the Centrosome and Nucleus}},
volume = {115},
year = {2003}
}
@article{Ahringer2003,
abstract = {Cell polarity is an essential feature of many animal cells. It is critical for epithelial formation and function, for correct partitioning of fate-determining molecules, and for individual cells to chemotax or grow in a defined direction. For some of these processes, the position and orientation of the mitotic spindle must be coupled to cell polarity for correct positioning of daughter cells and inheritance of localised molecules. Recent work in several different systems has led to the realisation that similar mechanisms dictate the establishment of polarity and subsequent spindle positioning in many animal cells. Microtubules and conserved PAR proteins are essential mediators of cell polarity, and mitotic spindle positioning depends on heterotrimeric G protein signalling and the microtubule motor protein dynein.},
author = {Ahringer, Julie},
doi = {10.1016/S0955-0674(02)00018-2},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}COCB{\_}15{\_}73.pdf:pdf},
isbn = {0955-0674},
issn = {09550674},
journal = {Current opinion in cell biology},
pages = {73--81},
pmid = {12517707},
title = {{Control of cell polarity and mitotic spindle positioning in animal cells.}},
volume = {15},
year = {2003}
}
@article{Reichenbach2003,
abstract = {Telomeres protect the eukaryotic chromosome ends from degradation and fusion. They are maintained by the ribonucleoprotein telomerase, the core of which is composed of a reverse transcriptase (TERT) and a RNA subunit. In the yeast Saccharomyces cerevisiae, a third critical telomerase subunit, the Ever Shorter Telomeres 1 (EST1) gene product, recruits or activates telomerase at the 3' end of telomeres. Est1p has so far only been known in budding yeast, and mechanisms that mediate telomerase access and activation in other eukaryotes have remained elusive. Here, we use iterative profile searches to identify homologs of yeast Est1p in a large variety of eukaryotes, including human. One of three human homologs, designated human EST1A (hEST1A), is shown to be associated with most or all active telomerase in HeLa cell extracts. Overexpression of hEST1A induces anaphase bridges due to chromosome-end fusions, and telomeric DNA persists at the fusion points. Thus, overexpression of hEST1A affects telomere capping. The identification of EST1 homologs in a large variety of eukaryotes may indicate that the mechanisms of telomere extension are more conserved than anticipated previously.},
author = {Reichenbach, Patrick and H{\"{o}}ss, Matthias and Azzalin, Claus M and Nabholz, Markus and Bucher, Philipp and Lingner, Joachim},
doi = {10.1016/S},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}CurrBiol{\_}13{\_}1029.pdf:pdf},
isbn = {0960-9822 (Print)$\backslash$n0960-9822 (Linking)},
issn = {0960-9822},
journal = {Current biology : CB},
keywords = {Amino Acid Sequence,Animals,Blotting,Chromosome Mapping,Fibroblasts,Fibroblasts: metabolism,Fluorescent Dyes,Hela Cells,Humans,Indoles,Molecular Sequence Data,Phylogeny,Saccharomyces cerevisiae Proteins,Saccharomyces cerevisiae Proteins: genetics,Saccharomyces cerevisiae Proteins: metabolism,Telomerase,Telomerase: genetics,Telomerase: metabolism,Telomere,Telomere: metabolism,Up-Regulation,Up-Regulation: genetics,Western},
number = {7},
pages = {568--74},
pmid = {12676087},
title = {{A human homolog of yeast Est1 associates with telomerase and uncaps chromosome ends when overexpressed.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/12676087},
volume = {13},
year = {2003}
}
@article{Reichenbach2003a,
abstract = {Telomeres protect the eukaryotic chromosome ends from degradation and fusion. They are maintained by the ribonucleoprotein telomerase, the core of which is composed of a reverse transcriptase (TERT) and a RNA subunit. In the yeast Saccharomyces cerevisiae, a third critical telomerase subunit, the Ever Shorter Telomeres 1 (EST1) gene product, recruits or activates telomerase at the 3' end of telomeres. Est1p has so far only been known in budding yeast, and mechanisms that mediate telomerase access and activation in other eukaryotes have remained elusive. Here, we use iterative profile searches to identify homologs of yeast Est1p in a large variety of eukaryotes, including human. One of three human homologs, designated human EST1A (hEST1A), is shown to be associated with most or all active telomerase in HeLa cell extracts. Overexpression of hEST1A induces anaphase bridges due to chromosome-end fusions, and telomeric DNA persists at the fusion points. Thus, overexpression of hEST1A affects telomere capping. The identification of EST1 homologs in a large variety of eukaryotes may indicate that the mechanisms of telomere extension are more conserved than anticipated previously.},
author = {Reichenbach, Patrick and H{\"{o}}ss, Matthias and Azzalin, Claus M and Nabholz, Markus and Bucher, Philipp and Lingner, Joachim},
doi = {10.1016/S},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}CurrBiol{\_}13{\_}1311.pdf:pdf},
isbn = {0960-9822 (Print)$\backslash$n0960-9822 (Linking)},
issn = {0960-9822},
journal = {Current biology : CB},
keywords = {Amino Acid Sequence,Animals,Blotting,Chromosome Mapping,Fibroblasts,Fibroblasts: metabolism,Fluorescent Dyes,Hela Cells,Humans,Indoles,Molecular Sequence Data,Phylogeny,Saccharomyces cerevisiae Proteins,Saccharomyces cerevisiae Proteins: genetics,Saccharomyces cerevisiae Proteins: metabolism,Telomerase,Telomerase: genetics,Telomerase: metabolism,Telomere,Telomere: metabolism,Up-Regulation,Up-Regulation: genetics,Western},
number = {7},
pages = {568--74},
pmid = {12676087},
title = {{A human homolog of yeast Est1 associates with telomerase and uncaps chromosome ends when overexpressed.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/12676087},
volume = {13},
year = {2003}
}
@article{Reichenbach2003b,
abstract = {Telomeres protect the eukaryotic chromosome ends from degradation and fusion. They are maintained by the ribonucleoprotein telomerase, the core of which is composed of a reverse transcriptase (TERT) and a RNA subunit. In the yeast Saccharomyces cerevisiae, a third critical telomerase subunit, the Ever Shorter Telomeres 1 (EST1) gene product, recruits or activates telomerase at the 3' end of telomeres. Est1p has so far only been known in budding yeast, and mechanisms that mediate telomerase access and activation in other eukaryotes have remained elusive. Here, we use iterative profile searches to identify homologs of yeast Est1p in a large variety of eukaryotes, including human. One of three human homologs, designated human EST1A (hEST1A), is shown to be associated with most or all active telomerase in HeLa cell extracts. Overexpression of hEST1A induces anaphase bridges due to chromosome-end fusions, and telomeric DNA persists at the fusion points. Thus, overexpression of hEST1A affects telomere capping. The identification of EST1 homologs in a large variety of eukaryotes may indicate that the mechanisms of telomere extension are more conserved than anticipated previously.},
author = {Reichenbach, Patrick and H{\"{o}}ss, Matthias and Azzalin, Claus M and Nabholz, Markus and Bucher, Philipp and Lingner, Joachim},
doi = {10.1016/S},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}CurrBiol{\_}13{\_}1499.pdf:pdf},
isbn = {0960-9822 (Print)$\backslash$n0960-9822 (Linking)},
issn = {0960-9822},
journal = {Current biology : CB},
keywords = {Amino Acid Sequence,Animals,Blotting,Chromosome Mapping,Fibroblasts,Fibroblasts: metabolism,Fluorescent Dyes,Hela Cells,Humans,Indoles,Molecular Sequence Data,Phylogeny,Saccharomyces cerevisiae Proteins,Saccharomyces cerevisiae Proteins: genetics,Saccharomyces cerevisiae Proteins: metabolism,Telomerase,Telomerase: genetics,Telomerase: metabolism,Telomere,Telomere: metabolism,Up-Regulation,Up-Regulation: genetics,Western},
number = {7},
pages = {568--74},
pmid = {12676087},
title = {{A human homolog of yeast Est1 associates with telomerase and uncaps chromosome ends when overexpressed.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/12676087},
volume = {13},
year = {2003}
}
@article{Pothof2003,
author = {Pothof, Joris and Haaften, Gijs Van and Thijssen, Karen and Kamath, Ravi S and Fraser, Andrew G and Ahringer, Julie and Plasterk, Ronald H A and Tijsterman, Marcel},
doi = {10.1101/gad.1060703.GENES},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}GeneDev{\_}17{\_}443.pdf:pdf},
journal = {Genes {\&} Development},
pages = {443--448},
title = {{Identification of genes that protect the}},
year = {2003}
}
@article{Kamath2003,
abstract = {In Caenorhabditis elegans, introduction of double-stranded RNA (dsRNA) results in the specific inactivation of an endogenous gene with corresponding sequence; this technique is known as RNA interference (RNAi). It has previously been shown that RNAi can be performed by direct microinjection of dsRNA into adult hermaphrodite worms, by soaking worms in a solution of dsRNA, or by feeding worms Escherichia coli expressing target-gene dsRNA. We have developed a simple optimized protocol exploiting this third mode of dsRNA introduction, RNAi by feeding, which allows rapid and effective analysis of gene function in C. elegans. Furthermore, we have constructed a library of bacterial strains corresponding to roughly 86{\%} of the estimated 19,000 predicted genes in C. elegans, and we have used it to perform genome-wide analyses of gene function. This library is publicly available, reusable resource allowing for rapid large-scale RNAi experiments. We have used this library to perform genome-wide analyses of gene function in C. elegans. Here, we describe the protocols used for bacterial library construction and for high-throughput screening in C. elegans using RNAi by feeding. ?? 2003 Elsevier Science (USA). All rights reserved.},
author = {Kamath, Ravi S. and Ahringer, Julie},
doi = {10.1016/S1046-2023(03)00050-1},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}Methods{\_}30{\_}313.pdf:pdf},
isbn = {4412233340},
issn = {10462023},
journal = {Methods},
number = {4},
pages = {313--321},
pmid = {12828945},
title = {{Genome-wide RNAi screening in Caenorhabditis elegans}},
volume = {30},
year = {2003}
}
@article{LeeSSFraserAGKamathRSAhringerJRuvkinG2003,
abstract = {We report a systematic RNA interference (RNAi) screen of 5,690 Caenorhabditis elegans genes for gene inactivations that increase lifespan. We found that genes important for mitochondrial function stand out as a principal group of genes affecting C. elegans lifespan. A classical genetic screen identified a mutation in the mitochondrial leucyl-tRNA synthetase gene (lrs-2) that impaired mitochondrial function and was associated with longer-lifespan. The long-lived worms with impaired mitochondria had lower ATP content and oxygen consumption, but differential responses to free-radical and other stresses. These data suggest that the longer lifespan of C. elegans with compromised mitochrondria cannot simply be assigned to lower free radical production and suggest a more complex coupling of metabolism and longevity. },
author = {{Lee SS  Fraser AG, Kamath RS, Ahringer J, Ruvkin G}, Lee R Y},
doi = {10.1038/ng1056},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}NatGen{\_}33{\_}40.pdf:pdf},
issn = {1061-4036},
journal = {Nature Genetics},
number = {1},
pages = {40--48},
title = {{A systematic RNAi screen identifies a critical role for mitochondria in C. elegans longevity.}},
volume = {33},
year = {2003}
}
@article{Kamath2003a,
abstract = {A principal challenge currently facing biologists is how to connect the complete DNA sequence of an organism to its development and behaviour. Large-scale targeted-deletions have been successful in defining gene functions in the single-celled yeast Saccharomyces cerevisiae, but comparable analyses have yet to be performed in an animal. Here we describe the use of RNA interference to inhibit the function of approximately 86{\%} of the 19,427 predicted genes of C. elegans. We identified mutant phenotypes for 1,722 genes, about two-thirds of which were not previously associated with a phenotype. We find that genes of similar functions are clustered in distinct, multi-megabase regions of individual chromosomes; genes in these regions tend to share transcriptional profiles. Our resulting data set and reusable RNAi library of 16,757 bacterial clones will facilitate systematic analyses of the connections among gene sequence, chromosomal location and gene function in C. elegans.},
author = {Kamath, Ravi S and Fraser, Andrew G and Dong, Yan and Poulin, Gino and Durbin, Richard and Gotta, Monica and Kanapin, Alexander and {Le Bot}, Nathalie and Moreno, Sergio and Sohrmann, Marc and Welchman, David P and Zipperlen, Peder and Ahringer, Julie},
doi = {10.1038/nature01278},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}Nature{\_}421{\_}231.pdf:pdf},
isbn = {0028-0836 (Print)$\backslash$n0028-0836 (Linking)},
issn = {0028-0836},
journal = {Nature},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans: genetics,Computational Biology,Evolution, Molecular,Genes, Helminth,Genes, Helminth: genetics,Genome,Genomics,Helminth Proteins,Helminth Proteins: chemistry,Helminth Proteins: genetics,Humans,Multigene Family,Multigene Family: genetics,Phenotype,Protein Structure, Tertiary,RNA Interference,RNA, Helminth,RNA, Helminth: genetics,RNA, Helminth: metabolism,Transcription, Genetic,Transcription, Genetic: genetics,X Chromosome,X Chromosome: genetics},
number = {6920},
pages = {231--7},
pmid = {12529635},
title = {{Systematic functional analysis of the Caenorhabditis elegans genome using RNAi.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/12529635},
volume = {421},
year = {2003}
}
@article{Ashrafi2003,
abstract = {Regulation of body fat storage involves signalling between centres that regulate feeding in the brain and sites of fat storage and use in the body. Here we describe an assay for analysing fat storage and mobilization in living Caenorhabditis elegans. By using RNA-mediated interference (RNAi) to disrupt the expression of each of the 16,757 worm genes, we have systematically screened the C. elegans genome for genes necessary for normal fat storage. We identify 305 gene inactivations that cause reduced body fat and 112 gene inactivations that cause increased fat storage. Analysis of the fat-reducing gene inactivations in insulin, serotonin and tubby signalling mutants of C. elegans, which have increased body fat, identifies a core set of fat regulatory genes as well as pathway-specific fat regulators. Many of the newly identified worm fat regulatory genes have mammalian homologues, some of which are known to function in fat regulation. Other C. elegans fat regulatory genes that are conserved across animal phylogeny, but have not previously been implicated in fat storage, may point to ancient and universal features of fat storage regulation, and identify targets for treating obesity and its associated diseases.},
author = {Ashrafi, Kaveh and Chang, Francesca Y and Watts, Jennifer L and Fraser, Andrew G and Kamath, Ravi S and Ahringer, Julie and Ruvkun, Gary},
doi = {10.1038/nature01279},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}Nature{\_}421{\_}268.pdf:pdf},
isbn = {0028-0836 (Print)$\backslash$r0028-0836 (Linking)},
issn = {0028-0836},
journal = {Nature},
number = {6920},
pages = {268--272},
pmid = {12529643},
title = {{Genome-wide RNAi analysis of Caenorhabditis elegans fat regulatory genes.}},
volume = {421},
year = {2003}
}
@article{Simmer2003,
abstract = {RNA-mediated interference (RNAi) is a method to inhibit gene function by introduction of double-stranded RNA (dsRNA). Recently, an RNAi library was constructed that consists of bacterial clones expressing dsRNA, corresponding to nearly 90{\%} of the 19,427 predicted genes of C. elegans. Feeding of this RNAi library to the standard wild-type laboratory strain Bristol N2 detected phenotypes for approximately 10{\%} of the corresponding genes. To increase the number of genes for which a loss-of-function phenotype can be detected, we undertook a genome-wide RNAi screen using the rrf-3 mutant strain, which we found to be hypersensitive to RNAi. Feeding of the RNAi library to rrf-3 mutants resulted in additional loss-of-function phenotypes for 393 genes, increasing the number of genes with a phenotype by 23{\%}. These additional phenotypes are distributed over different phenotypic classes. We also studied interexperimental variability in RNAi results and found persistent levels of false negatives. In addition, we used the RNAi phenotypes obtained with the genome-wide screens to systematically clone seven existing genetic mutants with visible phenotypes. The genome-wide RNAi screen using rrf-3 significantly increased the functional data on the C. elegans genome. The resulting dataset will be valuable in conjunction with other functional genomics approaches, as well as in other model organisms.},
author = {Simmer, Femke and Moorman, Celine and van der Linden, Alexander M and Kuijk, Ewart and van den Berghe, Peter V E and Kamath, Ravi S and Fraser, Andrew G and Ahringer, Julie and Plasterk, Ronald H A},
doi = {10.1371/journal.pbio.0000012},
file = {:Users/przemol/Documents/Mendeley Desktop/2003{\_}PLoS{\_}1{\_}E12.pdf:pdf},
isbn = {1545-7885 (Electronic)},
issn = {1545-7885},
journal = {PLoS biology},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans Proteins: metabolism,Caenorhabditis elegans: genetics,Chromosome Mapping,Chromosomes,Chromosomes: ultrastructure,Cloning, Molecular,Gene Expression Regulation,Gene Library,Genes, Helminth,Genetic Techniques,Genome,Genomics,Models, Biological,Mutation,Phenotype,RNA Interference,RNA, Double-Stranded,RNA, Double-Stranded: chemistry,Species Specificity},
number = {1},
pages = {E12},
pmid = {14551910},
title = {{Genome-wide RNAi of C. elegans using the hypersensitive rrf-3 strain reveals novel gene functions.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=212692{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {1},
year = {2003}
}
@article{Simmer2002,
abstract = {RNA interference (RNAi) is a broadly used reverse genetics method in C. elegans. Unfortunately, RNAi does not inhibit all genes. We show that loss of function of a putative RNA-directed RNA polymerase (RdRP) of C. elegans, RRF-3, results in a substantial enhancement of sensitivity to RNAi in diverse tissues. This is particularly striking in the nervous system; neurons that are generally refractory to RNAi in a wild-type genetic background can respond effectively to interference in an rrf-3 mutant background. These data provide the first indication of physiological negative modulation of the RNAi response and implicate an RdRP-related factor in this effect. The rrf-3 strain can be useful to study genes that, in wild-type, do not show a phenotype after RNAi, and it is probably the strain of choice for genome-wide RNAi screens.},
author = {Simmer, Femke and Tijsterman, Marcel and Parrish, Susan and Koushika, Sandhya P and Nonet, Michael L and Fire, Andrew and Ahringer, Julie and Plasterk, Ronald H a},
doi = {10.1016/S0960-9822(02)01041-2},
file = {:Users/przemol/Documents/Mendeley Desktop/2002{\_}CurrBiol{\_}12{\_}1317.pdf:pdf},
isbn = {0960-9822 (Print)},
issn = {0960-9822},
journal = {Current biology : CB},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans: enzymology,Caenorhabditis elegans: genetics,Phenotype,RNA Interference,RNA Replicase,RNA Replicase: deficiency,RNA Replicase: genetics,RNA Replicase: metabolism,RNA, Double-Stranded,RNA, Double-Stranded: genetics},
number = {15},
pages = {1317--9},
pmid = {12176360},
title = {{Loss of the putative RNA-directed RNA polymerase RRF-3 makes C. elegans hypersensitive to RNAi.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/12176360},
volume = {12},
year = {2002}
}
@article{Esmaeili2002,
abstract = {Locomotory activity is defined by the specification of motoneurone subtypes. In the nematode, C. elegans, DA and DB motoneurones innervate dorsal muscles and function to induce movement in the backwards or forwards direction, respectively. These two neurone classes express separate sets of genes and extend axons with oppositely directed trajectories; anterior (DA) versus posterior (DB). The DA-specific homeoprotein UNC-4 interacts with UNC-37/Groucho to repress the DB gene, acr-5 (nicotinic acetylcholine receptor subunit). We show that the C. elegans even-skipped-like homoedomain protein, VAB-7, coordinately regulates different aspects of the DB motoneurone fate, in part by repressing unc-4. Wild-type DB motoneurones express VAB-7, have posteriorly directed axons, express ACR-5 and lack expression of the homeodomain protein UNC-4. In a vab-7 mutant, ectopic UNC-4 represses acr-5 and induces an anteriorly directed DB axon trajectory. Thus, vab-7 indirectly promotes DB-specific gene expression and posteriorly directed axon outgrowth by preventing UNC-4 repression of DB differentiation. Ectopic expression of VAB-7 also induces DB traits in an unc-4-independent manner, suggesting that VAB-7 can act through a parallel pathway. This work supports a model in which a complementary pair of homeodomain transcription factors (VAB-7 and UNC-4) specifies differences between DA and DB neurones through inhibition of the alternative fates. The recent findings that Even-skipped transcriptional repressor activity specifies neurone identity and axon guidance in the mouse and Drosophila motoneurone circuit points to an ancient origin for homeoprotein-dependent mechanisms of neuronal differentiation in the metazoan nerve cord.},
author = {Esmaeili, Behrooz and Ross, Jennifer M and Neades, Cara and Miller, David M and Ahringer, Julie},
file = {:Users/przemol/Documents/Mendeley Desktop/2002{\_}Dev{\_}129{\_}853.pdf:pdf},
isbn = {0950-1991 (Print)$\backslash$n0950-1991 (Linking)},
issn = {0950-1991},
journal = {Development (Cambridge, England)},
keywords = {axons,c,elegans,motoneurone identity,vab-7},
number = {4},
pages = {853--862},
pmid = {11861469},
title = {{The C. elegans even-skipped homologue, vab-7, specifies DB motoneurone identity and axon trajectory.}},
volume = {129},
year = {2002}
}
@article{Garigan2002,
abstract = {The genetic analysis of life span has revealed many interesting genes and pathways; however, our understanding of aging has been limited by the lack of a way to assay the aging process itself. Here we show that the tissues of aging worms have a characteristic appearance that is easy to recognize and quantify using Nomarski optics. We have used this assay to determine whether life-span mutations affect the rate of aging, to identify animals that age more rapidly than normal, and to infer the cause of death in C. elegans. Mutations that reduce insulin/IGF-1 signaling double the life span of C. elegans, and we find that tissue decline is slowed in these mutants. Thus this endocrine system appears to influence the rate at which tissues age. This effect extends even to the germline, which is the only mitotically active tissue in the adult. We find that Nomarski microscopy also allows a ready distinction between short-lived mutants that age more rapidly than normal and those that are simply sick, and we have identified an RNAi clone that confers a dramatic rapid-aging phenotype. This clone encodes the C. elegans heat-shock factor (HSF), a transcription factor that regulates the response to heat and oxidative stress. This suggests that heat-shock proteins, many of which act as chaperones, may function in normal animals to slow the rate of aging. Finally, we have identified a cause of death of C. elegans: namely, proliferating bacteria. This suggests that increased susceptibility to bacterial infections contributes to mortality in these animals, just as it does in humans.},
author = {Garigan, Delia and Hsu, Ao Lin and Fraser, Andrew G. and Kamath, Ravi S. and Abringet, Julie and Kenyon, Cynthia},
file = {:Users/przemol/Documents/Mendeley Desktop/2002{\_}Genetics{\_}161{\_}1101.pdf:pdf},
isbn = {0016-6731 (Print)},
issn = {00166731},
journal = {Genetics},
number = {3},
pages = {1101--1112},
pmid = {12136014},
title = {{Genetic analysis of tissue aging in Caenorhabditis elegans: A role for heat-shock factor and bacterial proliferation}},
volume = {161},
year = {2002}
}
@article{Dillin2002,
author = {Dillin, Andrew and Hsu, Ao-lin and Arantes-oliveira, Nuno and Lehrer-graiwer, Joshua and Hsin, Honor and Fraser, Andrew G},
doi = {10.1126/science.1077780},
file = {:Users/przemol/Documents/Mendeley Desktop/2002{\_}Science{\_}298{\_}2398.pdf:pdf},
issn = {00368075},
journal = {Science},
number = {DECEMBER},
pages = {2398--2401},
title = {{Rates of Behavior and Aging Speci ed by Mitochondrial Function During Development}},
volume = {298},
year = {2002}
}
@article{Gotta2001,
abstract = {The anterior-posterior axis in Caenorhabditis elegans is determined by the sperm and leads to the asymmetric localisation of PAR (partitioning-defective) proteins, which are critical for polarity. New findings demonstrate that sperm asters play a critical role and suggest models for how PAR asymmetry is established. In addition, studies of blastomere fate determination and heterotrimeric G proteins have started to uncover how initial polarity may be translated into the asymmetric distribution of maternal proteins and the control of spindle position.},
author = {Gotta, M and Ahringer, J},
doi = {S0959-437X(00)00206-9 [pii]},
file = {:Users/przemol/Documents/Mendeley Desktop/2001{\_}COGD{\_}11{\_}367.pdf:pdf},
isbn = {0959-437X (Print)$\backslash$r0959-437X (Linking)},
issn = {0959-437X},
journal = {Current Opinion In Genetics {\&} Development},
number = {4},
pages = {367--373},
pmid = {11448622},
title = {{Axis determination in C-elegans: initiating and transducing polarity}},
url = {http://apps.isiknowledge.com/InboundService.do?product=WOS{\&}action=retrieve{\&}SrcApp=Papers{\&}UT=000170069200002{\&}SID=T1Mcj4NaE82AnoHE@Gi{\&}SrcAuth=mekentosj{\&}mode=FullRecord{\&}customersID=mekentosj{\&}DestFail=http://access.isiproducts.com/custom{\_}images/wok{\_}failed{\_}auth.html$\backslash$npapers2://publication/uuid/AFCAD339-D8BC-4E54-BF59-8632255B1164},
volume = {11},
year = {2001}
}
@article{Gotta2001a,
abstract = {Background: Generation of asymmetry in the one-cell embryo of C. elegans establishes the anterior-posterior axis (A-P), and is necessary for the proper identity of early blastomeres. Conserved PAR proteins are asymmetrically distributed and are required for the generation of this early asymmetry. The small G protein Cdc42 is a key regulator of polarity in other systems, and recently it has been shown to interact with the mammalian homolog of PAR-6. The function of Cdc42 in C. elegans had not yet been investigated, however. Results: Here, we show that C. elegans cdc-42 plays an essential role in the polarity of the one-cell embryo and the proper localization of PAR proteins. Inhibition of cdc-42 using RNA interference results in embryos with a phenotype that is nearly identical to par-3, par-6, and pkc-3 mutants, and asymmetric localization of these and other PAR proteins is lost. We further show that C. elegans CDC-42 physically interacts with PAR-6 in a yeast two-hybrid system, consistent with data on the interaction of human homologs. Conclusions: Our results show that CDC-42 acts in concert with the PAR proteins to control the polarity of the C. elegans embryo, and provide evidence that the interaction of CDC-42 and the PAR-3/PAR-6/PKC-3 complex has been evolutionarily conserved as a functional unit.},
author = {Gotta, Monica and Abraham, Mary C. and Ahringer, Julie},
doi = {10.1016/S0960-9822(01)00142-7},
file = {:Users/przemol/Documents/Mendeley Desktop/2001{\_}CurrBiol{\_}11{\_}482.pdf:pdf},
isbn = {0960-9822 (Print)$\backslash$r0960-9822 (Linking)},
issn = {09609822},
journal = {Current Biology},
number = {7},
pages = {482--488},
pmid = {11412997},
title = {{CDC-42 controls early cell polarity and spindle orientation in C. elegans}},
volume = {11},
year = {2001}
}
@article{Zipperlen2001,
abstract = {Early embryonic development involves complex events such as the regulation of cell division and the establishment of embryonic polarity. To identify genes involved in these events, we collected four-dimensional time-lapse video recordings of the first three cell divisions and analysed terminal phenotypes after RNA interference of 147 embryonic lethal genes previously identified in a systematic screen of Caenorhabditis elegans chromosome I. Over half gave defects in early processes such as meiosis, the assembly or position of the first mitotic spindle, cytokinesis, and proper nuclear positioning. For some phenotypic classes, the majority of genes are involved in a shared biochemical process. In addition, we identified loss-of-function phenotypes for genes of unknown function, but for which homologues exist in other organisms, shedding light on the function of these uncharacterized genes. When applied to the whole genome, this approach should identify the vast majority of genes required for early cell processes, paving the way for a greatly improved understanding of these processes and their regulation at the molecular level.},
author = {Zipperlen, P and Fraser, A G and Kamath, R S and Martinez-Campos, M and Ahringer, J},
doi = {10.1093/emboj/20.15.3984},
file = {:Users/przemol/Documents/Mendeley Desktop/2001{\_}EMBOJ{\_}20{\_}3984.pdf:pdf},
isbn = {0261-4189 (Print)$\backslash$r0261-4189 (Linking)},
issn = {0261-4189},
journal = {The EMBO journal},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans: embryology,Caenorhabditis elegans: genetics,Gene Expression,Genes, Helminth,Germ Cells,Humans,Microscopy, Video,Phenotype,RNA, Helminth},
number = {15},
pages = {3984--92},
pmid = {11483502},
title = {{Roles for 147 embryonic lethal genes on C.elegans chromosome I identified by RNA interference and video microscopy.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=149177{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {20},
year = {2001}
}
@article{Kamath2001,
abstract = {BACKGROUND: In Caenorhabditis elegans, injection of double-stranded RNA (dsRNA) results in the specific inactivation of genes containing homologous sequences, a technique termed RNA-mediated interference (RNAi). It has previously been shown that RNAi can also be achieved by feeding worms Escherichia coli expressing dsRNA corresponding to a specific gene; this mode of dsRNA introduction is conventionally considered to be less efficient than direct injection, however, and has therefore seen limited use, even though it is considerably less labor-intensive.$\backslash$n$\backslash$nRESULTS: Here we present an optimized feeding method that results in phenotypes at least as strong as those produced by direct injection of dsRNA for embryonic lethal genes, and stronger for genes with post-embryonic phenotypes. In addition, the interference effect generated by feeding can be titrated to uncover a series of hypomorphic phenotypes informative about the functions of a given gene. Using this method, we screened 86 random genes on consecutive cosmids and identified functions for 13 new genes. These included two genes producing an uncoordinated phenotype (a previously uncharacterized POU homeodomain gene, ceh-6, and a gene encoding a MADS-box protein) and one gene encoding a novel protein that results in a high-incidence-of-males phenotype.$\backslash$n$\backslash$nCONCLUSIONS: RNAi by feeding can provide significant information about the functions of an individual gene beyond that provided by injection. Moreover, it can be used for special applications for which injection or the use of mutants is sometimes impracticable (for example, titration, biochemistry and large-scale screening). Thus, RNAi by feeding should make possible new experimental approaches for the use of genomic sequence information.},
author = {Kamath, R S and Martinez-Campos, M and Zipperlen, P and Fraser, A G and Ahringer, J},
doi = {10.1186/gb-2000-2-1-research0002},
file = {:Users/przemol/Documents/Mendeley Desktop/2001{\_}GenmBiol{\_}2{\_}r2.pdf:pdf},
isbn = {1465-6914 (Electronic)$\backslash$n1465-6914 (Linking)},
issn = {1465-6914},
journal = {Genome biology},
keywords = {Animals,Caenorhabditis elegans,Caenorhabditis elegans: drug effects,Caenorhabditis elegans: genetics,Escherichia coli,Escherichia coli: genetics,Genes, Helminth,Genes, Helminth: genetics,Genetic Vectors,Genetic Vectors: administration {\&} dosage,Genetic Vectors: genetics,Microinjections,Phenotype,RNA, Double-Stranded,RNA, Double-Stranded: administration {\&} dosage,RNA, Double-Stranded: genetics,RNA, Helminth,RNA, Helminth: drug effects,RNA, Helminth: genetics,RNA, Helminth: metabolism,Transfection,Transfection: methods,Transformation, Genetic},
number = {1},
pages = {RESEARCH0002},
pmid = {11178279},
title = {{Effectiveness of specific RNA-mediated interference through ingested double-stranded RNA in Caenorhabditis elegans.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=17598{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {2},
year = {2001}
}
@article{Gotta2001b,
abstract = {Correct placement and orientation of the mitotic spindle is essential for segregation of localized components and positioning of daughter cells. Although these processes are important in many cells, few factors that regulate spindle placement are known. Previous work has shown that GPB-1, the Gbeta subunit of a heterotrimeric G protein, is required for orientation of early cell division axes in C. elegans embryos. Here we show that GOA-1 (a Galphao) and the related GPA-16 are the functionally redundant Galpha subunits and that GPC-2 is the relevant Ggamma subunit that is required for spindle orientation in the early embryo. We show that Galpha and Gbetagamma are involved in controlling distinct microtubule-dependent processes. Gbetagamma is important in regulating migration of the centrosome around the nucleus and hence in orientating the mitotic spindle. Galpha is required for asymmetric spindle positioning in the one-celled embryo.},
author = {Gotta, M and Ahringer, J},
doi = {10.1038/35060092},
file = {:Users/przemol/Documents/Mendeley Desktop/2001{\_}NatCB{\_}3{\_}297.pdf:pdf},
isbn = {1465-7392 (Print)$\backslash$r1465-7392 (Linking)},
issn = {1465-7392},
journal = {Nature cell biology},
number = {3},
pages = {297--300},
pmid = {11231580},
title = {{Distinct roles for Galpha and Gbetagamma in regulating spindle position and orientation in Caenorhabditis elegans embryos.}},
volume = {3},
year = {2001}
}
@article{Solari2000,
abstract = {Chromatin-modifying complexes are important for transcriptional control, but their roles in the regulation of development are poorly understood. Here, we show that components of the nucleosome remodelling and histone deacetylase (NURD) complex [1-5] antagonise vulval development, which is induced by the Ras signal transduction pathway. In three of the six equivalent vulval precursor cells, the Ras pathway is active, leading to the production of vulval fates [6]; in the remaining three, the Ras pathway is inhibited and vulval fates repressed. Inhibition of Ras signaling occurs in part through the action of the synthetic multivulval (synMuv) genes, which comprise two functionally redundant pathways (synMuvA and synMuvB) [7]. We found that five Caenorhabditis elegans members of the NURD chromatin remodelling complex inhibit vulval development through both the synMuvA and synMuvB pathways (hda-1, rba-1, lin-53, chd-3 and chd-4); a further two members, the MTA1-related genes egr-1 and egl-27, act only in the synMuvA pathway. We propose that the synMuvA and synMuvB pathways function redundantly to recruit or activate a core NURD complex, which then represses vulval developmental target genes by local histone deacetylation. These results emphasise the importance of chromatin regulation in developmental decisions. Furthermore, inhibition of Ras signaling suggests a possible link between NURD function and cancer.},
author = {Solari, Florence and Ahringer, Julie},
doi = {10.1016/S0960-9822(00)00343-2},
file = {:Users/przemol/Documents/Mendeley Desktop/2000{\_}CurrBiol{\_}10{\_}223.pdf:pdf},
isbn = {0960-9822 (Print)$\backslash$r0960-9822 (Linking)},
issn = {0960-9822},
journal = {Current Biology},
number = {4},
pages = {223--226},
pmid = {10704416},
title = {{NURD-complex genes antagonise Ras-induced vulval development in Caenorhabditis elegans}},
volume = {10},
year = {2000}
}
@article{Fraser2000,
abstract = {Complete genomic sequence is known for two multicellular eukaryotes, the nematode Caenorhabditis elegans and the fruit fly Drosophila melanogaster, and it will soon be known for humans. However, biological function has been assigned to only a small proportion of the predicted genes in any animal. Here we have used RNA-mediated interference (RNAi) to target nearly 90{\%} of predicted genes on C. elegans chromosome I by feeding worms with bacteria that express double-stranded RNA. We have assigned function to 13.9{\%} of the genes analysed, increasing the number of sequenced genes with known phenotypes on chromosome I from 70 to 378. Although most genes with sterile or embryonic lethal RNAi phenotypes are involved in basal cell metabolism, many genes giving post-embryonic phenotypes have conserved sequences but unknown function. In addition, conserved genes are significantly more likely to have an RNAi phenotype than are genes with no conservation. We have constructed a reusable library of bacterial clones that will permit unlimited RNAi screens in the future; this should help develop a more complete view of the relationships between the genome, gene function and the environment.},
author = {Fraser, a G and Kamath, R S and Zipperlen, P and Martinez-Campos, M and Sohrmann, M and Ahringer, J},
doi = {10.1038/35042517},
file = {:Users/przemol/Documents/Mendeley Desktop/2000{\_}Nature{\_}408{\_}325.pdf:pdf},
isbn = {0028-0836},
issn = {0028-0836},
journal = {Nature},
number = {6810},
pages = {325--330},
pmid = {11099033},
title = {{Functional genomic analysis of C. elegans chromosome I by systematic RNA interference.}},
volume = {408},
year = {2000}
}
@article{Ahringer2000,
abstract = {Transcription repression mediated through histone deacetylase (HDAC) complexes is widespread, and mechanisms by which HDAC complexes act have been revealed by extensive studies in vitro and in cell culture. However, until recently, little has been known about the developmental roles of histone deacetylation. Mutants now exist for a number of members of the two major HDAC complexes (NuRD and SIN3) and some associated proteins. The emerging picture is that these complexes have specific functions in development, rather than being required for most cellular processes.},
author = {Ahringer, Julie},
doi = {10.1016/S0168-9525(00)02066-7},
file = {:Users/przemol/Documents/Mendeley Desktop/2000{\_}TIG{\_}16{\_}351.pdf:pdf},
isbn = {0168-9525 (Print)$\backslash$n0168-9525 (Linking)},
issn = {01689525},
journal = {Trends in Genetics},
number = {1997},
pages = {351--356},
pmid = {10904264},
title = {{NuRD and SIN3: Histone deacetylase complexes in development}},
volume = {16},
year = {2000}
}
@article{Solari1999,
abstract = {We show here that two functionally redundant Caenorhabditis elegans genes, egl-27 and egr-1, have a fundamental role in embryonic patterning. When both are inactivated, cells in essentially all regions of the embryo fail to be properly organised. Tissue determination and differentiation are unaffected and many zygotic patterning genes are expressed normally, including HOX genes. However, hlh-8, a target of the HOX gene mab-5, is not expressed. egl-27 and egr-1 are members of a gene family that includes MTA1, a human gene with elevated expression in metastatic carcinomas. MTA1 is a component of a protein complex with histone deacetylase and nucleosome remodelling activities. We propose that EGL-27 and EGR-1 function as part of a chromatin regulatory complex required for the function of regional patterning genes.},
author = {Solari, Florence and Bateman, Alex and Ahringer, Julie},
file = {:Users/przemol/Documents/Mendeley Desktop/1999{\_}Dev{\_}126{\_}2483.pdf:pdf},
isbn = {0950-1991 (Print)},
issn = {0950-1991},
journal = {Development (Cambridge, England)},
keywords = {caenorhabditis,deacetylation,elm1,elm2,histone,nucleosome remodelling,nurd,sant,vab-7},
number = {11},
pages = {2483--2494},
pmid = {10226007},
title = {{The Caenorhabditis elegans genes egl-27 and egr-1 are similar to MTA1, a member of a chromatin regulatory complex, and are redundantly required for embryonic patterning.}},
volume = {126},
year = {1999}
}
@article{Gallegos1998,
abstract = {The fem-3 sex-determining gene is repressed post-transcriptionally via a regulatory element in its 3' untranslated region (UTR) to achieve the switch from spermatogenesis to oogenesis in the Caenorhabditis elegans hermaphrodite germ line. In this paper, we investigate the fem-3 3' UTR control in somatic tissues using transgenic reporter assays, and we also identify six genes essential for this control. First, we find that a reporter transgene bearing a wild-type fem-3 3' UTR is repressed in somatic tissues, whereas one bearing a mutant fem-3 3' UTR is derepressed. Moreover, control by mutant 3' UTRs is temperature sensitive as predicted from the temperature sensitivity of the fem-3 gain-of-function (gf) mutations. Secondly, we find a fem-3 3' UTR RNA-binding activity in somatic tissues, in addition to the previously reported germ-line-specific binding by FBF. Thirdly, we find that each of six genes, mog-1-mog-6, is required for repression by the fem-3 3' UTR. Therefore, the mog genes not only affect the sperm/oocyte switch in the germ line, but also function in somatic tissues. We suggest that the mog genes may encode components of a ubiquitous machinery that is used for fem-3 3' UTR-mediated repression and the sperm/oocyte switch.},
author = {Gallegos, M and Ahringer, J and Crittenden, S and Kimble, J},
doi = {10.1093/emboj/17.21.6337},
file = {:Users/przemol/Documents/Mendeley Desktop/1998{\_}EMBOJ{\_}17{\_}6337.pdf:pdf},
isbn = {0261-4189 (Print)},
issn = {0261-4189},
journal = {The EMBO journal},
keywords = {3' Untranslated Regions,3' Untranslated Regions: genetics,Animals,Animals, Genetically Modified,Caenorhabditis elegans,Caenorhabditis elegans Proteins,Caenorhabditis elegans: genetics,Gene Expression Regulation, Developmental,Gene Expression Regulation, Developmental: genetic,Genes, Helminth,Genes, Helminth: genetics,Genes, Reporter,Genes, Reporter: genetics,Helminth Proteins,Helminth Proteins: genetics,Histocytochemistry,Mutation,Mutation: genetics,Phenotype,RNA, Messenger,RNA, Messenger: genetics,RNA-Binding Proteins,RNA-Binding Proteins: genetics,Regulatory Sequences, Nucleic Acid,Regulatory Sequences, Nucleic Acid: genetics,Sex Determination Processes,Temperature},
number = {21},
pages = {6337--47},
pmid = {9799241},
title = {{Repression by the 3' UTR of fem-3, a sex-determining gene, relies on a ubiquitous mog-dependent control in Caenorhabditis elegans.}},
url = {http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1170958{\&}tool=pmcentrez{\&}rendertype=abstract},
volume = {17},
year = {1998}
}
@article{Ahringer1997,
abstract = {Caenorhabditis elegans will be the first multicellular animal to have its entire genome sequenced. This is not just good news for those currently working in the field, but also for those trying to understand the biology of more complex animals, including humans. C elegans is a relatively simple animal that is amenable to studies of genetics and developmental processes that are common to all animals, making this an attractive model in which to study basic processes that are altered in human disease. Powerful forward and reverse genetics mean that virtually any gene of interest can be studied at the functional level.},
author = {Ahringer, J},
file = {:Users/przemol/Documents/Mendeley Desktop/1997{\_}COGD{\_}7{\_}410.pdf:pdf},
issn = {0959-437X},
journal = {Current opinion in genetics {\&} development},
keywords = {Animals,Base Sequence,Caenorhabditis elegans,Caenorhabditis elegans: genetics,Genetic Techniques,Genetics,Genome,Human,Humans,Medical,Molecular Sequence Data,Nucleic Acid,Sequence Homology},
number = {3},
pages = {410--5},
pmid = {9229119},
title = {{Turn to the worm!}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/9229119},
volume = {7},
year = {1997}
}
@article{Zwaal1996,
abstract = {Heterotrimeric G proteins are signal-transducing molecules activated by seven transmembrane domain receptors. In C. elegans, gpb-1 encodes the sole Gbeta subunit; therefore, its inactivation should affect all heterotrimeric G protein signaling. When maternal but no zygotic gpb-1 protein (GPB-1) is present, development proceeds until the first larval stage, but these larvae show little muscle activity and die soon after hatching. When, however, the maternal contribution of GPB-1 is also reduced, spindle orientations in early cell divisions are randomized. Cell positions in these embryos are consequently abnormal, and the embryos die with the normal number of cells and well-differentiated but abnormally distributed tissues. These results indicate that maternal G proteins are important for orientation of early cell division axes, possibly by coupling a membrane signal to centrosome position.},
author = {Zwaal, R R and Ahringer, J and van Luenen, H G A M and Rushforth, A and Anderson, P and Plasterk, R H A},
file = {:Users/przemol/Documents/Mendeley Desktop/1996{\_}Cell{\_}86{\_}619.pdf:pdf},
issn = {0092-8674},
journal = {Cell},
pages = {619--629},
pmid = {8752216},
title = {{G protein are required for spatial orientation of early cell cleavages in C. elegans embryos}},
volume = {86},
year = {1996}
}
@article{Goedert1996,
abstract = {Tau, MAP2 and MAP4 are structural microtubule-associated proteins (MAPs) that promote the assembly and stability of microtubules. They share three or four imperfect tandem repeats of an amino acid motif, which is involved in the binding to microtubules. All sequences to data containing this motif are of mammalian origin. We report here the cloning and functional characterisation of a new member of this family of proteins from the nematode Caenorhabditis elegans. This protein exists as two isoforms of 413 and 453 amino acids with four or five tandem repeats that are 50{\%} identical to the tau/MAP2/MAP4 repeats. Both isoforms bind to microtubules and promote microtubule assembly, with the five-repeat isoform being more effective at promoting assembly than the four-repeat isoform. When expressed in COS cells, the five-repeat isoform co-localises with microtubules and induces the formation of microtubule bundles, whereas its expression in Sf9 cells leads to the extension of long unipolar processes. In view of its length, amino acid sequence and functional characteristics, we have named this invertebrate structural MAP 'Protein with Tau-Like Repeats' (PTL-1). In C. elegans PTL-1 is expressed in two places known to require microtubule function. It is first seen in the embryonic epidermis, when circumferentially oriented microtubules help to distribute forces generated during elongation. Later, it is found in mechanosensory neurons which contain unusual 15 protofilament microtubules required for the response to touch. These findings indicate that MAPs of the tau/MAP2/MAP4 family are found throughout much of the animal kingdom, where they may play a role in specialised processes requiring microtubules.},
author = {Goedert, M and Baur, C P and Ahringer, J and Jakes, R and Hasegawa, M and Spillantini, M G and Smith, M J and Hill, F},
file = {:Users/przemol/Documents/Mendeley Desktop/1996{\_}JCS{\_}109{\_}2661.pdf:pdf},
isbn = {0021-9533 (Print)$\backslash$n0021-9533 (Linking)},
issn = {0021-9533},
journal = {Journal of cell science},
keywords = {can be distinguished,map1a and map1b share,map2,map4,microtubule-associated,partial,protein,protein with tau-like repeats,sequence similarity and are,synthesised as polyproteins,tau},
pages = {2661--2672},
pmid = {8937984},
title = {{PTL-1, a microtubule-associated protein with tau-like repeats from the nematode Caenorhabditis elegans.}},
volume = {109 ( Pt 1},
year = {1996}
}


</pre>
  	<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
		<script type="text/javascript" src="build/bib-list.js"></script>
		<script type="text/javascript">
		    $(document).ready(function() {
        	bibtexify("#bibtex", "pubTable", {'tweet': 'vkaravir'});
    		});
		</script>
	</body>
</html>
